<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> Publications | Nhat Truong Pham </title> <meta name="author" content="Nhat Truong Pham"> <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design. "> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/icon.png?1e5a7fb652b1868ab49bdd3c8fe588a6"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://nhattruongpham.github.io/publications/"> <script src="/assets/js/theme.js?9a0c749ec5240d9cda97bc72359a72c0"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>initTheme();</script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> <span class="font-weight-bold">Nhat Truong</span> Pham </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">Home </a> </li> <li class="nav-item "> <a class="nav-link" href="/news/">News </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">Publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/collaborators/">Collaborators </a> </li> <li class="nav-item "> <a class="nav-link" href="/mentees/">Mentees </a> </li> <li class="nav-item "> <a class="nav-link" href="/repositories/">Repositories </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">CV </a> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Tools </a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item " href="/molecules/">Molecules</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/dna/">DNA</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/rna/">RNA</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/peptide/">Peptide</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/protein/">Protein</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/microbiome/">Microbiome</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/imaging/">Imaging</a> </div> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Gallery </a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item " href="/skkurs24/">SKKU-RS-24</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/ijcai24/">IJCAI-24</a> <div class="dropdown-divider"></div> <a class="dropdown-item " href="/acl24/">ACL-24</a> </div> </li> <li class="nav-item"> <button id="search-toggle" title="Search" onclick="openSearchModal()"> <span class="nav-link">ctrl k <i class="ti ti-search"></i></span> </button> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">Publications</h1> <p class="post-description"></p> </header> <article> <p> (†) denotes equal contribution </p> <p> (*) denotes correspondance </p> <p> <span style="display: inline-block; width: 15px; height: 15px; background-color: #600;"></span> denotes journal </p> <p> <span style="display: inline-block; width: 15px; height: 15px; background-color: #215d42;"></span> denotes conference </p> <p> <span style="display: inline-block; width: 15px; height: 15px; background-color: #0d93bf;"></span> denotes preprint </p> <script src="/assets/js/bibsearch.js?1bc438ca9037884cc579601c09afd847" type="module"></script> <p><input type="text" id="bibsearch" spellcheck="false" autocomplete="off" class="search bibsearch-form-input" placeholder="Search documents with keywords"></p> <div class="publications"> <h2 class="bibliography">2025</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://www.nature.com/srep/" rel="external nofollow noopener" target="_blank">Sci Rep</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/MemoCMT.jpg" sizes="200px"></source> <img src="/assets/img/publication_preview/MemoCMT.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="MemoCMT.jpg" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="khan2025memocmt" class="col-sm-8"> <div class="title"> <b><i>MemoCMT:</i></b> multimodal emotion recognition using cross-modal transformer-based feature fusion</div> <div class="author"> <a href="https://scholar.google.com/citations?hl=en&amp;user=uEZhRWAAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Mustaqeem Khan<sup>†</sup></a> , <a href="https://scholar.google.com/citations?hl=en&amp;user=NKbwDD8AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Phuong-Nam Tran<sup>†</sup></a> , <em><b style="color: #FFA500;">Nhat Truong Pham<sup>†</sup></b></em> , <a href="https://scholar.google.com/citations?hl=en&amp;user=VcOjgngAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Abdulmotaleb El Saddik</a> , and <a href="https://scholar.google.com/citations?hl=en&amp;user=nlonFkwAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Alice Othmani</a> </div> <div class="periodical"> <em><i><a href="https://www.nature.com/srep/" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Scientific Reports</a></i></em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.nature.com/articles/s41598-025-89202-x" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/MemoCMT.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a> <a href="https://github.com/tpnam0901/MemoCMT/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41598-025-89202-x" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:A8cqit5AE6sC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-0-4285F4?logo=googlescholar&amp;labelColor=beige" alt="0 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Speech emotion recognition has seen a surge in transformer models, which excel at understanding the overall message by analyzing long-term patterns in speech. However, these models come at a computational cost. In contrast, convolutional neural networks are faster but struggle with capturing these long-range relationships. Our proposed system, MemoCMT, tackles this challenge using a novel “cross-modal transformer” (CMT). This CMT can effectively analyze local and global speech features and their corresponding text. To boost efficiency, MemoCMT leverages recent advancements in pre-trained models: HuBERT extracts meaningful features from the audio, while BERT analyzes the text. The core innovation lies in how the CMT component utilizes and integrates these audio and text features. After this integration, different fusion techniques are applied before final emotion classification. Experiments show that MemoCMT achieves impressive performance, with the CMT using min aggregation achieving the highest unweighted accuracy (UW-Acc) of 81.33% and 91.93%, and weighted accuracy (W-Acc) of 81.85% and 91.84% respectively on benchmark IEMOCAP and ESD corpora. The results of our system demonstrate the generalization capacity and robustness for real-world industrial applications. Moreover, the implementation details of MemoCMT are publicly available at <a href="https://github.com/tpnam0901/MemoCMT/" rel="external nofollow noopener" target="_blank">https://github.com/tpnam0901/MemoCMT/</a> for reproducibility purposes.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">khan2025memocmt</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{MemoCMT: multimodal emotion recognition using cross-modal transformer-based feature fusion}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Khan, Mustaqeem and Tran, Phuong-Nam and Pham, Nhat Truong and El Saddik, Abdulmotaleb and Othmani, Alice}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Scientific Reports}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{15}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{5473}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Nature Publishing Group UK London}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41598-025-89202-x}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://www.sciencedirect.com/journal/journal-of-molecular-biology" rel="external nofollow noopener" target="_blank">JMB</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/DOGpred.jpeg" sizes="200px"></source> <img src="/assets/img/publication_preview/DOGpred.jpeg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="DOGpred.jpeg" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="lee2025dogpred" class="col-sm-8"> <div class="title"> <b><i>DOGpred:</i></b> A Novel Deep Learning Framework for Accurate Identification of Human O-linked Threonine Glycosylation Sites</div> <div class="author"> Ki Wook Lee<sup>†</sup> , <em><b style="color: #FFA500;">Nhat Truong Pham<sup>†</sup></b></em> , Hye Jung Min , Hyun Woo Park , Ji Won Lee , Han-En Lo , Na Young Kwon , Jimin Seo , Illia Shaginyan , Heeje Cho , <a href="https://scholar.google.com/citations?hl=en&amp;user=0EAV03MAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Leyi Wei</a> , <a href="https://scholar.google.com/citations?hl=en&amp;user=0vkenbwAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Balachandran Manavalan</a> , and <a href="https://scholar.google.com/citations?hl=en&amp;user=DkVnJ-wAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Young-Jun Jeon</a> </div> <div class="periodical"> <em><i><a href="https://www.sciencedirect.com/journal/journal-of-molecular-biology" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Journal of Molecular Biology</a></i></em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/pii/S0022283625000439" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://github.com/JeonRPM/DOGpred/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> <a href="https://doi.org/10.5281/zenodo.13141341" rel="external nofollow noopener" target="_blank"> <img src="https://zenodo.org/badge/doi/10.5281/zenodo.13141341.svg" alt="DOI"> </a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.jmb.2025.168977" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:MIg0yeAD4ggC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-1-4285F4?logo=googlescholar&amp;labelColor=beige" alt="1 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>O-linked glycosylation is a crucial post-transcriptional modification that regulates protein function and biological processes. Dysregulation of this process is associated with various diseases, underscoring the need to accurately identify O-linked glycosylation sites on proteins. Current experimental methods for identifying O-linked threonine glycosylation (OTG) sites are often complex and costly. Consequently, developing computational tools that predict these sites based on protein features is crucial. Such tools can complement experimental approaches, enhancing our understanding of the role of OTG dysregulation in diseases and uncovering potential therapeutic targets. In this study, we developed DOGpred, a deep learning-based predictor for precisely identifying human OTGs using high-latent feature representations. Initially, we extracted nine different conventional feature descriptors (CFDs) and nine pre-trained protein language model (PLM)-based embeddings. Notably, each feature was encoded as a 2D tensor, capturing both the sequential and inherent feature characteristics. Subsequently, we designed a stacked convolutional neural network (CNN) module to learn spatial feature representations from CFDs and a stacked recurrent neural network (RNN) module to learn temporal feature representations from PLM-based embeddings. These features were integrated using attention-based fusion mechanisms to generate high-level feature representations for final classification. Ablation analysis and independent tests demonstrated that the optimal model (DOGpred), employing a stacked 1D CNN and a stacked attention-based RNN module with cross-attention feature fusion, achieved the best performance on the training dataset and significantly outperformed machine learning-based single-feature models and state-of-the-art methods on independent datasets. Furthermore, DOGpred is publicly available at <a href="https://github.com/JeonRPM/DOGpred/" rel="external nofollow noopener" target="_blank">https://github.com/JeonRPM/DOGpred/</a> for free access and usage.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">lee2025dogpred</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{DOGpred: A Novel Deep Learning Framework for Accurate Identification of Human O-linked Threonine Glycosylation Sites}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Lee, Ki Wook and Pham, Nhat Truong and Min, Hye Jung and Park, Hyun Woo and Lee, Ji Won and Lo, Han-En and Kwon, Na Young and Seo, Jimin and Shaginyan, Illia and Cho, Heeje and Wei, Leyi and Manavalan, Balachandran and Jeon, Young-Jun}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Journal of Molecular Biology}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{168977}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.jmb.2025.168977}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://www.sciencedirect.com/journal/ocean-engineering" rel="external nofollow noopener" target="_blank">OE</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/PHBPB_Structure.png" sizes="200px"></source> <img src="/assets/img/publication_preview/PHBPB_Structure.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="PHBPB_Structure.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="anh2025hydraulic" class="col-sm-8"> <div class="title">Hydraulic performance and wave transmission through nature-inspired perforated hollow-base piles breakwater</div> <div class="author"> <a href="https://scholar.google.com/citations?hl=en&amp;user=4MfplgUAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Duong Tran Anh</a> , Trinh Cong Dan , <a href="https://scholar.google.com/citations?hl=en&amp;user=PYCmxMQAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Phong Nguyen Thanh</a> , Ahad Hasan Tanim , David Wright , <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , and Nguyen Anh Tien </div> <div class="periodical"> <em><i><a href="https://www.sciencedirect.com/journal/ocean-engineering" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Ocean Engineering</a></i></em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/pii/S0029801825001386" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.oceaneng.2025.120423" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:z6xuaG2dYH0C" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-0-4285F4?logo=googlescholar&amp;labelColor=beige" alt="0 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Nature-based Solutions (NbS) are critical to achieving long-term sustainable coastlines in deltaic coastlines, particularly susceptible to severe impacts under climate change. In recent years, coastal erosion has accelerated in the Vietnam Mekong Delta (VMD), particularly in the West Sea (from Ca Mau to Kien Giang). Due to the characteristics of muddy coastlines and eroded mangrove forests in this region, only a few engineering solutions have effectively reduced waves and promoted sediment accumulation to facilitate beach accretion. However, these solutions require significant investment and cannot be relocated or reused. Therefore, this study introduces a coastal mangrove-inspired Perforated Hollow-Base Pile Breakwater (PHBPB), which uses the principle of biomimicry by studying the ability of mangrove groups to protect the coastline, stimulate sediment deposition, and over time, restore mangrove forests in the VMD. The hydraulic parameters of PHBPB were tested in 260 scenarios using a physical model, which included varying parameters such as Crest width (B), Pile row numbers (n<sub>c</sub>), Crest freeboard (R<sub>c</sub>), and Wave parameters (H<sub>s</sub>, T<sub>p</sub>) in determining the capabilities of wave transmission, reflection, and wave dissipation. The results showed that the process of wave energy dissipation through Perforated Hollow-Base Breakwaters Without Piles (PHBBWP) is governed by three main parameters, including the relative water depth of the crest (R<sub>c</sub>/H<sub>m0,i</sub>), the relative width of the crest (B/H<sub>m0,i</sub>), and the wave slope at the structure location (s<sub>m</sub> = H<sub>m0,i</sub>/L<sub>m</sub>). In contrast, for PHBPB, the primary influencing factors are the relative submergence depth or the length of the submerged pile portion (R<sub>c</sub>/H<sub>m0,i</sub>) and the relative width of the pile system (X<sub>b</sub>/L<sub>m</sub>). The hollow structure is suitable for the soft, weak mud geology in this region with poor load-bearing capacity, allows for rapid installation, is reasonably cost-effective, and can be relocated and reused once regeneration has been completed in each stretch of coastline.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">anh2025hydraulic</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Hydraulic performance and wave transmission through nature-inspired perforated hollow-base piles breakwater}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Anh, Duong Tran and Dan, Trinh Cong and Thanh, Phong Nguyen and Tanim, Ahad Hasan and Wright, David and Pham, Nhat Truong and Tien, Nguyen Anh}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Ocean Engineering}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{322}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{120423}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.oceaneng.2025.120423}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://www.tandfonline.com/journals/thsj20" rel="external nofollow noopener" target="_blank">HSJ</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/VE_Runoff.jpg" sizes="200px"></source> <img src="/assets/img/publication_preview/VE_Runoff.jpg" class="preview z-depth-1 rounded" width="100%" height="auto" alt="VE_Runoff.jpg" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="le2025enhancing" class="col-sm-8"> <div class="title">Enhancing daily runoff prediction in hydropower basins with a voting ensemble model using historical data</div> <div class="author"> Ngoc Anh LE , <a href="https://scholar.google.com/citations?hl=en&amp;user=PYCmxMQAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Phong Nguyen Thanh</a> , <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , Le Quoc Huy , Son T. Mai , Duc Dung Do , Huy Anh Nguyen , and <a href="https://scholar.google.com/citations?hl=en&amp;user=4MfplgUAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Duong Tran Anh</a> </div> <div class="periodical"> <em><i><a href="https://www.tandfonline.com/journals/thsj20" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Hydrological Sciences Journal</a></i></em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="/assets/html/" class="btn btn-sm z-depth-0" role="button">HTML</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1080/02626667.2025.2461705" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-N/A-4285F4?logo=googlescholar&amp;labelColor=beige" alt="N/A Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Enhancing high-precision daily runoff prediction in hydropower basins is crucial for effective water management and flood early warning systems (FEWS). Additionally, evaluating the potential of machine learning (ML) models still needs room for improvement. Hence, this study initially assesses five individual ML models, including multilayer perceptron (MLP), support vector regressor (SVR), random forest (RF), extreme gradient boosting (XGB), and catboost regressor (CBR), to predict daily runoff. Subsequently, a voting ensemble (VE) model was developed to enhance the accuracy of the single models. The VE model further improves performance by reducing root mean squared error (RMSE) values by 1% to 6% (year-round), 2% to 14% (dry season), and up to 7% (flood season) compared to the individual ML models. SHAP (SHapley Additive exPlanations) analysis was also applied to interpret model predictions, highlighting the most influential features. Overall, the VE model significantly improves forecast precision, making it especially effective in hydropower-affected basins, and its predictions support flood modelling and FEWS in downstream areas.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">le2025enhancing</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Enhancing daily runoff prediction in hydropower basins with a voting ensemble model using historical data}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{LE, Ngoc Anh and Thanh, Phong Nguyen and Pham, Nhat Truong and Huy, Le Quoc and Mai, Son T. and Do, Duc Dung and Nguyen, Huy Anh and Anh, Duong Tran}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Hydrological Sciences Journal}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Taylor &amp; Francis}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1080/02626667.2025.2461705}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#0d93bf"> <div>Preprint</div> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/RSS_RM.png" sizes="200px"></source> <img src="/assets/img/publication_preview/RSS_RM.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="RSS_RM.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="yang2025advances" class="col-sm-8"> <div class="title">Advances in RNA secondary structure prediction and RNA modifications: Methods, data, and applications</div> <div class="author"> Shu Yang<sup>†</sup> , <em><b style="color: #FFA500;">Nhat Truong Pham<sup>†</sup></b></em> , <a href="https://scholar.google.com/citations?hl=en&amp;user=tROyBasAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Ziyang Li<sup>†</sup></a> , Jae Young Baik<sup>†</sup> , Joseph Lee , Tianhua Zhai , Weicheng Yu , Bojian Hou , Tianqi Shang , Weiqing He , <a href="https://scholar.google.com/citations?hl=en&amp;user=m_qvsgoAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Duy Duong-Tran</a> , <a href="https://scholar.google.com/citations?hl=en&amp;user=fmsV6nEAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Mayur Naik</a> , and <a href="https://scholar.google.com/citations?hl=en&amp;user=QnWpiskAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Li Shen</a> </div> <div class="periodical"> <em><i><a href="" style="color: #FF3636; text-decoration: underline dashed;"></a></i></em> 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://arxiv.org/abs/2501.04056" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">arXiv</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://arxiv.org/pdf/2501.04056" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> </div> <div class="badges"> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:SAZ1SQo2q1kC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-0-4285F4?logo=googlescholar&amp;labelColor=beige" alt="0 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Due to the hierarchical organization of RNA structures and their pivotal roles in fulfilling RNA functions, the formation of RNA secondary structure critically influences many biological processes and has thus been a crucial research topic. This review sets out to explore the computational prediction of RNA secondary structure and its connections to RNA modifications, which have emerged as an active domain in recent years. We first examine the progression of RNA secondary structure prediction methodology, focusing on a set of representative works categorized into thermodynamic, comparative, machine learning, and hybrid approaches. Next, we survey the advances in RNA modifications and computational methods for identifying RNA modifications, focusing on the prominent modification types. Subsequently, we highlight the interplay between RNA modifications and secondary structures, emphasizing how modifications such as m6A dynamically affect RNA folding and vice versa. In addition, we also review relevant data sources and provide a discussion of current challenges and opportunities in the field. Ultimately, we hope our review will be able to serve as a cornerstone to aid in the development of innovative methods for this emerging topic and foster therapeutic applications in the future.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">yang2025advances</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Advances in RNA secondary structure prediction and RNA modifications: Methods, data, and applications}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Yang, Shu and Pham, Nhat Truong and Li, Ziyang and Baik, Jae Young and Lee, Joseph and Zhai, Tianhua and Yu, Weicheng and Hou, Bojian and Shang, Tianqi and He, Weiqing and Duong-Tran, Duy and Naik, Mayur and Shen, Li}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://www.sciencedirect.com/journal/computers-in-biology-and-medicine" rel="external nofollow noopener" target="_blank">CIBM</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/XCT-COVID.png" sizes="200px"></source> <img src="/assets/img/publication_preview/XCT-COVID.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="XCT-COVID.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="pham2025leveraging" class="col-sm-8"> <div class="title">Leveraging deep transfer learning and explainable AI for accurate COVID-19 diagnosis: Insights from a multi-national chest CT scan study</div> <div class="author"> <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , Jinsol Ko , <a href="https://scholar.google.com/citations?hl=en&amp;user=W5BcBWoAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Masaud Shah</a> , <a href="https://scholar.google.com/citations?hl=en&amp;user=5WP7pOUAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Rajan Rakkiyappan</a> , <a href="https://scholar.google.com/citations?hl=en&amp;user=oM75QQMAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Hyun Goo Woo</a> , and <a href="https://scholar.google.com/citations?hl=en&amp;user=0vkenbwAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Balachandran Manavalan</a> </div> <div class="periodical"> <em><i><a href="https://www.sciencedirect.com/journal/computers-in-biology-and-medicine" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Computers in Biology and Medicine</a></i></em>, 2025 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/abs/pii/S0010482524015464" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://github.com/cbbl-skku-org/XCT-COVID/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> <a href="https://doi.org/10.5281/zenodo.12772023" rel="external nofollow noopener" target="_blank"> <img src="https://zenodo.org/badge/doi/10.5281/zenodo.12772023.svg" alt="DOI"> </a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.compbiomed.2024.109461" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:Wq2b2clWBLsC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-1-4285F4?logo=googlescholar&amp;labelColor=beige" alt="1 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>The COVID-19 pandemic has emerged as a global health crisis, impacting millions worldwide. Although chest computed tomography (CT) scan images are pivotal in diagnosing COVID-19, their manual interpretation by radiologists is time-consuming and potentially subjective. Automated computer-aided diagnostic (CAD) frameworks offer efficient and objective solutions. However, machine or deep learning methods often face challenges in their reproducibility due to underlying biases and methodological flaws. To address these issues, we propose XCT-COVID, an explainable, transferable, and reproducible CAD framework based on deep transfer learning to predict COVID-19 infection from CT scan images accurately. This is the first study to develop three distinct models within a unified framework by leveraging a previously unexplored large dataset and two widely used smaller datasets. We employed five known convolutional neural network architectures, both with and without pretrained weights, on the larger dataset. We optimized hyperparameters through extensive grid search and 5-fold cross-validation (CV), significantly enhancing the model performance. Experimental results from the larger dataset showed that the VGG16 architecture (XCT-COVID-L) with pretrained weights consistently outperformed other architectures, achieving the best performance, on both 5-fold CV and independent test. When evaluated with the external datasets, XCT-COVID-L performed well with data with similar distributions, demonstrating its transferability. However, its performance significantly decreased on smaller datasets with lower-quality images. To address this, we developed other models, XCT-COVID-S1 and XCT-COVID-S2, specifically for the smaller datasets, outperforming existing methods. Moreover, eXplainable Artificial Intelligence (XAI) analyses were employed to interpret the models’ functionalities. For prediction and reproducibility purposes, the implementation of XCT-COVID is publicly accessible at <a href="https://github.com/cbbl-skku-org/XCT-COVID/" rel="external nofollow noopener" target="_blank">https://github.com/cbbl-skku-org/XCT-COVID/</a>.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">pham2025leveraging</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Leveraging deep transfer learning and explainable AI for accurate COVID-19 diagnosis: Insights from a multi-national chest CT scan study}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Pham, Nhat Truong and Ko, Jinsol and Shah, Masaud and Rakkiyappan, Rajan and Woo, Hyun Goo and Manavalan, Balachandran}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Computers in Biology and Medicine}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{185}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{109461}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2025}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.compbiomed.2024.109461}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> <h2 class="bibliography">2024</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#0d93bf"> <div>Preprint</div> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/QTSeg.png" sizes="200px"></source> <img src="/assets/img/publication_preview/QTSeg.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="QTSeg.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="tran2024qtseg" class="col-sm-8"> <div class="title"> <b><i>QTSeg:</i></b> A Query Token-Based Dual-Mix Attention Framework with Multi-Level Feature Distribution for Medical Image Segmentation</div> <div class="author"> <a href="https://scholar.google.com/citations?hl=en&amp;user=NKbwDD8AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Phuong-Nam Tran</a> , <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , <a href="https://scholar.google.com/citations?hl=en&amp;user=2UKP440AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Duc Ngoc Minh Dang</a> , <a href="https://scholar.google.com/citations?hl=en&amp;user=0ATpZl0AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Eui-Nam Huh</a> , and <a href="https://scholar.google.com/citations?hl=en&amp;user=oKANWloAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Choong Seon Hong</a> </div> <div class="periodical"> <em><i><a href="" style="color: #FF3636; text-decoration: underline dashed;"></a></i></em> 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://arxiv.org/abs/2412.17241" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">arXiv</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://arxiv.org/pdf/2412.17241" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> <a href="https://github.com/tpnam0901/QTSeg/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> </div> <div class="badges"> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:qE4H1tSSYIIC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-0-4285F4?logo=googlescholar&amp;labelColor=beige" alt="0 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Medical image segmentation plays a crucial role in assisting healthcare professionals with accurate diagnoses and enabling automated diagnostic processes. Traditional convolutional neural networks (CNNs) often struggle with capturing long-range dependencies, while transformer-based architectures, despite their effectiveness, come with increased computational complexity. Recent efforts have focused on combining CNNs and transformers to balance performance and efficiency, but existing approaches still face challenges in achieving high segmentation accuracy while maintaining low computational costs. Furthermore, many methods underutilize the CNN encoder’s capability to capture local spatial information, concentrating primarily on mitigating long-range dependency issues. To address these limitations, we propose QTSeg, a novel architecture for medical image segmentation that effectively integrates local and global information. QTSeg features a dual-mix attention decoder designed to enhance segmentation performance through: (1) a cross-attention mechanism for improved feature alignment, (2) a spatial attention module to capture long-range dependencies, and (3) a channel attention block to learn inter-channel relationships. Additionally, we introduce a multi-level feature distribution module, which adaptively balances feature propagation between the encoder and decoder, further boosting performance. Extensive experiments on five publicly available datasets covering diverse segmentation tasks, including lesion, polyp, breast cancer, cell, and retinal vessel segmentation, demonstrate that QTSeg outperforms state-of-the-art methods across multiple evaluation metrics while maintaining lower computational costs.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">tran2024qtseg</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{QTSeg: A Query Token-Based Dual-Mix Attention Framework with Multi-Level Feature Distribution for Medical Image Segmentation}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Tran, Phuong-Nam and Pham, Nhat Truong and Dang, Duc Ngoc Minh and Huh, Eui-Nam and Hong, Choong Seon}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#0d93bf"> <div>Preprint</div> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/MMF_SER_Review.png" sizes="200px"></source> <img src="/assets/img/publication_preview/MMF_SER_Review.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="MMF_SER_Review.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="nguyen5063214multi" class="col-sm-8"> <div class="title">Multi-modal fusion in speech emotion recognition: A comprehensive review of methods and technologies</div> <div class="author"> Nhut Minh Nguyen , Thanh Trung Nguyen , <a href="https://scholar.google.com/citations?hl=en&amp;user=NKbwDD8AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Phuong-Nam Tran</a> , <a href="https://scholar.google.com/citations?hl=en&amp;user=an_4IJkAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Chee Peng Lim</a> , <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , and <a href="https://scholar.google.com/citations?hl=en&amp;user=2UKP440AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Duc Ngoc Minh Dang</a> </div> <div class="periodical"> <em><i><a href="" style="color: #FF3636; text-decoration: underline dashed;"></a></i></em> 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://papers.ssrn.com/sol3/papers.cfm?abstract_id=5063214" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">SSRN</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://github.com/aita-lab/awesome-multimodal-fusion-emotion-recognition/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> </div> <div class="badges"> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:qE4H1tSSYIIC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-0-4285F4?logo=googlescholar&amp;labelColor=beige" alt="0 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Speech emotion recognition (SER) plays a crucial role in human-computer interaction, enhancing numerous applications such as virtual assistants, healthcare monitoring, and customer support by identifying and interpreting emotions conveyed through spoken language. While single-modality SER systems demonstrate notable simplicity and computational efficiency, excelling in extracting critical features like vocal prosody and linguistic content, there is a pressing need to improve their performance in challenging conditions, such as noisy environments and the handling of ambiguous expressions or incomplete information. These challenges underscore the necessity of transitioning to multi-modal approaches, which integrate complementary data sources to achieve more robust and accurate emotion detection. With advancements in artificial intelligence, especially in neural networks and deep learning, many studies have employed advanced deep learning and feature fusion techniques to enhance SER performance. This review synthesizes comprehensive publications from 2020 to 2024, exploring prominent multi-modal fusion strategies, including early fusion, late fusion, deep fusion, and hybrid fusion methods, while also examining data representation, data translation, attention mechanisms, and graph-based fusion technologies. We assess the effectiveness of various fusion techniques across standard SER datasets, highlighting their performance in diverse tasks and addressing challenges related to data alignment, noise management, and computational demands. Additionally, we explore potential future directions for enhancing multi-modal SER systems, emphasizing scalability and adaptability in real-world applications. This survey aims to contribute to the advancement of multi-modal SER and to inform researchers about effective fusion strategies for developing more responsive and emotion-aware systems.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">nguyen5063214multi</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Multi-modal fusion in speech emotion recognition: A comprehensive review of methods and technologies}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Nguyen, Nhut Minh and Nguyen, Thanh Trung and Tran, Phuong-Nam and Lim, Chee Peng and Pham, Nhat Truong and Dang, Duc Ngoc Minh}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://www.sciencedirect.com/journal/journal-of-molecular-biology" rel="external nofollow noopener" target="_blank">JMB</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/MST-m6A.png" sizes="200px"></source> <img src="/assets/img/publication_preview/MST-m6A.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="MST-m6A.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="su2024mst" class="col-sm-8"> <div class="title"> <b><i>MST-m6A:</i></b> A Novel Multi-Scale Transformer-based Framework for Accurate Prediction of m6A Modification Sites Across Diverse Cellular Contexts</div> <div class="author"> Qiaosen Su , Le Thi Phan , <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , <a href="https://scholar.google.com/citations?hl=en&amp;user=0EAV03MAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Leyi Wei</a> , and <a href="https://scholar.google.com/citations?hl=en&amp;user=0vkenbwAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Balachandran Manavalan</a> </div> <div class="periodical"> <em><i><a href="https://www.sciencedirect.com/journal/journal-of-molecular-biology" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Journal of Molecular Biology</a></i></em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/pii/S0022283624004868" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://github.com/cbbl-skku-org/MST-m6A/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> <a href="https://doi.org/10.5281/zenodo.12741567" rel="external nofollow noopener" target="_blank"> <img src="https://zenodo.org/badge/doi/10.5281/zenodo.12741567.svg" alt="DOI"> </a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.jmb.2024.168856" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:C33y2ycGS3YC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-2-4285F4?logo=googlescholar&amp;labelColor=beige" alt="2 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>N6-methyladenosine (m6A) modification, a prevalent epigenetic mark in eukaryotic cells, is crucial in regulating gene expression and RNA metabolism. Accurately identifying m6A modification sites is essential for understanding their functions within biological processes and the intricate mechanisms that regulate them. Recent advances in high-throughput sequencing technologies have enabled the generation of extensive datasets characterizing m6A modification sites at single-nucleotide resolution, leading to the development of computational methods for identifying m6A RNA modification sites. However, most current methods focus on specific cell lines, limiting their generalizability and practical application across diverse biological contexts. To address the limitation, we propose MST-m6A, a novel approach for identifying m6A modification sites with higher accuracy across various cell lines and tissues. MST-m6A utilizes a multi-scale transformer-based architecture, employing dual k-mer tokenization to capture rich feature representations and global contextual information from RNA sequences at multiple levels of granularity. These representations are then effectively combined using a channel fusion mechanism and further processed by a convolutional neural network to enhance prediction accuracy. Rigorous validation demonstrates that MST-m6A significantly outperforms conventional machine learning models, deep learning models, and state-of-the-art predictors. We anticipate that the high precision and cross-cell-type adaptability of MST-m6A will provide valuable insights into m6A biology and facilitate advancements in related fields. The proposed approach is available at <a href="https://github.com/cbbl-skku-org/MST-m6A/" rel="external nofollow noopener" target="_blank">https://github.com/cbbl-skku-org/MST-m6A/</a> for prediction and reproducibility purposes.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">su2024mst</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{MST-m6A: A Novel Multi-Scale Transformer-based Framework for Accurate Prediction of m6A Modification Sites Across Diverse Cellular Contexts}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Su, Qiaosen and Phan, Le Thi and Pham, Nhat Truong and Wei, Leyi and Manavalan, Balachandran}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Journal of Molecular Biology}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{168856}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.jmb.2024.168856}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#215d42"> <a href="https://www.acmmmasia.org/" rel="external nofollow noopener" target="_blank">ACM-MMAsia</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/HuBERT-CLAP.png" sizes="200px"></source> <img src="/assets/img/publication_preview/HuBERT-CLAP.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="HuBERT-CLAP.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="nguyen2024hubert" class="col-sm-8"> <div class="title"> <b><i>HuBERT-CLAP:</i></b> Contrastive Learning-Based Multimodal Emotion Recognition using Self-Alignment Approach</div> <div class="author"> Long H. Nguyen<sup>†</sup> , <em><b style="color: #FFA500;">Nhat Truong Pham<sup>†</sup></b></em> , <a href="https://scholar.google.com/citations?hl=en&amp;user=uEZhRWAAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Mustaqeem Khan</a> , <a href="https://scholar.google.com/citations?hl=en&amp;user=nlonFkwAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Alice Othmani</a> , and <a href="https://scholar.google.com/citations?hl=en&amp;user=VcOjgngAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Abdulmotaleb El Saddik</a> </div> <div class="periodical"> <em>In <i><a href="https://www.acmmmasia.org/" style="color: #00ab37; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Proceedings of the 6th ACM International Conference on Multimedia in Asia</a></i></em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://dl.acm.org/doi/10.1145/3696409.3700183" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/HuBERT-CLAP.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a> <a href="https://github.com/oggyfaker/HuBERT-CLAP/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1145/3696409.3700183" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:qE4H1tSSYIIC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-0-4285F4?logo=googlescholar&amp;labelColor=beige" alt="0 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>A breakthrough in deep learning has led to improvements in speech emotion recognition (SER), but these studies tend to process fixed-length segments, resulting in degraded performance. Therefore, multimodal approaches that combine audio and text features improve SER but lack modality alignment. In this study, we introduce HuBERT-CLAP, a contrastive language-audio self-alignment pre-training framework for SER to address the aforementioned issue. Initially, we employ CLIP to train a contrastive self-alignment model using HuBERT for audio and BERT/DistilBERT for text to extract discriminative cues from the input sequences and map informative features from text to audio features. Additionally, HuBERT in the pre-trained HuBERT-CLAP undergoes partial fine-tuning to enhance the effectiveness in predicting emotional states. Furthermore, we evaluated our model on the IEMOCAP dataset, where it outperformed the non-pre-training model, achieving a weighted accuracy of 77.22%. Our source code is publicly available at <a href="https://github.com/oggyfaker/HuBERT-CLAP/" rel="external nofollow noopener" target="_blank">https://github.com/oggyfaker/HuBERT-CLAP/</a> for reproducible purposes.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@inproceedings</span><span class="p">{</span><span class="nl">nguyen2024hubert</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{HuBERT-CLAP: Contrastive Learning-Based Multimodal Emotion Recognition using Self-Alignment Approach}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Nguyen, Long H. and Pham, Nhat Truong and Khan, Mustaqeem and Othmani, Alice and El Saddik, Abdulmotaleb}</span><span class="p">,</span>
  <span class="na">booktitle</span> <span class="p">=</span> <span class="s">{Proceedings of the 6th ACM International Conference on Multimedia in Asia}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1--6}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1145/3696409.3700183}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#215d42"> <a href="https://ictc.org/" rel="external nofollow noopener" target="_blank">ICTC</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/FL_U-Net.png" sizes="200px"></source> <img src="/assets/img/publication_preview/FL_U-Net.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="FL_U-Net.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="le2024federated" class="col-sm-8"> <div class="title">Federated Learning with U-Net for Brain Tumor Segmentation: Impact of Client Numbers and Data Distribution</div> <div class="author"> <a href="https://scholar.google.com/citations?hl=en&amp;user=l-bJWlcAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Thu Thuy Le</a> , <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , <a href="https://scholar.google.com/citations?hl=en&amp;user=NKbwDD8AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Phuong-Nam Tran</a> , and <a href="https://scholar.google.com/citations?hl=en&amp;user=2UKP440AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Duc Ngoc Minh Dang</a> </div> <div class="periodical"> <em>In <i><a href="https://ictc.org/" style="color: #00ab37; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">2024 15th International Conference on Information and Communication Technology Convergence (ICTC)</a></i></em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://ieeexplore.ieee.org/document/10826674" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1109/ICTC62082.2024.10826674" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:-6RzNnnwWf8C" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-0-4285F4?logo=googlescholar&amp;labelColor=beige" alt="0 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Brain tumor segmentation plays a crucial role in diagnosis and treatment planning. However, sharing patient data for training deep learning models raises privacy concerns. In this paper, we propose a federated learning (FL) approach that utilizes a U-Net architecture for brain tumor segmentation. We evaluate the performance of federated U-Net models across different data distribution scenarios and varying numbers of clients. Specifically, we compare the effectiveness of two FL methods: Federated Averaging (FedAvg) and Federated Stochastic Gradient Descent (FedSGD). Through experiments conducted on the BraTS dataset, we observe that as the number of clients increases, the overall performance of the models tends to decrease. Moreover, we find that skewed data distribution often outperforms equal data division. Additionally, we consistently observe that FedAvg yields superior results compared to FedSGD. Our proposed approach enables hospitals to collaboratively train models on their local data without directly sharing sensitive information. This preserves patient privacy while ensuring accurate tumor segmentation. The results of our study underscore the significance of strategic data distribution in FL environments and provide valuable insights for optimizing FL strategies in medical imaging applications.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@inproceedings</span><span class="p">{</span><span class="nl">le2024federated</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Federated Learning with U-Net for Brain Tumor Segmentation: Impact of Client Numbers and Data Distribution}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Le, Thu Thuy and Pham, Nhat Truong and Tran, Phuong-Nam and Dang, Duc Ngoc Minh}</span><span class="p">,</span>
  <span class="na">booktitle</span> <span class="p">=</span> <span class="s">{2024 15th International Conference on Information and Communication Technology Convergence ICTC}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{2048--2053}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">organization</span> <span class="p">=</span> <span class="s">{IEEE}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1109/ICTC62082.2024.10826674}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#215d42"> <a href="https://ictc.org/" rel="external nofollow noopener" target="_blank">ICTC</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/VNese-TrafficSign.png" sizes="200px"></source> <img src="/assets/img/publication_preview/VNese-TrafficSign.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="VNese-TrafficSign.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="tran2024towards" class="col-sm-8"> <div class="title">Towards Real-time Vietnamese Traffic Sign Recognition on Embedded Systems</div> <div class="author"> <a href="https://scholar.google.com/citations?hl=en&amp;user=NKbwDD8AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Phuong-Nam Tran</a> , <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , Nam Pham Van Hai , Duc Tai Phan , Tuan Cuong Nguyen , and <a href="https://scholar.google.com/citations?hl=en&amp;user=2UKP440AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Duc Ngoc Minh Dang</a> </div> <div class="periodical"> <em>In <i><a href="https://ictc.org/" style="color: #00ab37; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">2024 15th International Conference on Information and Communication Technology Convergence (ICTC)</a></i></em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://ieeexplore.ieee.org/document/10827558" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1109/ICTC62082.2024.10827558" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:T_ojBgVMvoEC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-0-4285F4?logo=googlescholar&amp;labelColor=beige" alt="0 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>In recent years, AI development has brought many significant changes in various aspects of our daily lives. Integrating AI technology into various applications has revolutionized multiple domains, and one particularly vital area is traffic sign recognition, which significantly enhances driver safety. This paper presents an approach to traffic sign recognition specifically designed for the Jetson Nano 2GB device. By utilizing the YOLOv8 Nano model, the proposed approach achieves a remarkable frame rate of up to 32 frames per second (FPS). To optimize inference speed on Jetson with limited memory, the approach incorporates TensorRT and quantization techniques. In addition, this paper introduces a dataset called the Vietnamese Traffic Sign Detection Database 100 (VTSDB100). This dataset is an extension of the VTSDB46 dataset and encompasses a comprehensive collection of 100 different classes of traffic signs. These signs were captured in diverse locations within Ho Chi Minh City, Vietnam, providing a rich and diverse dataset for training and evaluating traffic sign recognition models. An extensive experiment and analysis are also conducted using various object detection methods on the VTSDB100 dataset. The findings highlight the potential of deploying the proposed approach on resource-constrained devices and provide valuable insights for further research and development in the field of AI-powered driver safety systems.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@inproceedings</span><span class="p">{</span><span class="nl">tran2024towards</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Towards Real-time Vietnamese Traffic Sign Recognition on Embedded Systems}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Tran, Phuong-Nam and Pham, Nhat Truong and Pham Van Hai, Nam and Phan, Duc Tai and Nguyen, Tuan Cuong and Dang, Duc Ngoc Minh}</span><span class="p">,</span>
  <span class="na">booktitle</span> <span class="p">=</span> <span class="s">{2024 15th International Conference on Information and Communication Technology Convergence ICTC}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1--6}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">organization</span> <span class="p">=</span> <span class="s">{IEEE}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1109/ICTC62082.2024.10827558}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#215d42"> <a href="https://language-plus-molecules.github.io/" rel="external nofollow noopener" target="_blank">LPM@ACL</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/Mol2Lang-VLM.png" sizes="200px"></source> <img src="/assets/img/publication_preview/Mol2Lang-VLM.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="Mol2Lang-VLM.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="tran2024mollangvlm" class="col-sm-8"> <div class="title"> <b><i>Mol2Lang-VLM:</i></b> Vision- and Text-Guided Generative Pre-trained Language Models for Advancing Molecule Captioning through Multimodal Fusion</div> <div class="author"> <a href="https://scholar.google.com/citations?hl=en&amp;user=kz_chQ4AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Duong Thanh Tran<sup>†</sup></a> , <em><b style="color: #FFA500;">Nhat Truong Pham<sup>†</sup></b></em> , <a href="https://scholar.google.com/citations?hl=en&amp;user=-aEoZCgAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Nguyen Doan Hieu Nguyen</a> , and <a href="https://scholar.google.com/citations?hl=en&amp;user=0vkenbwAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Balachandran Manavalan</a> </div> <div class="periodical"> <em>In <i><a href="https://language-plus-molecules.github.io/" style="color: #00ab37; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Proceedings of the 1st Workshop on Language + Molecules (L+M 2024)</a></i></em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://aclanthology.org/2024.langmol-1.12/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://aclanthology.org/2024.langmol-1.12.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> <a href="https://github.com/nhattruongpham/mol-lang-bridge/tree/mol2lang" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> </div> <div class="badges"> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:AYInfyleIOsC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-1-4285F4?logo=googlescholar&amp;labelColor=beige" alt="1 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>This paper introduces Mol2Lang-VLM, an enhanced method for refining generative pre-trained language models for molecule captioning using multimodal features to achieve more accurate caption generation. Our approach leverages the encoder and decoder blocks of the Transformer-based architecture by introducing third sub-layers into both. Specifically, we insert sub-layers in the encoder to fuse features from SELFIES strings and molecular images, while the decoder fuses features from SMILES strings and their corresponding descriptions. Moreover, cross multi-head attention is employed instead of common multi-head attention to enable the decoder to attend to the encoder’s output, thereby integrating the encoded contextual information for better and more accurate caption generation. Performance evaluation on the CheBI-20 and L+M-24 benchmark datasets demonstrates Mol2Lang-VLM’s superiority, achieving higher accuracy and quality in caption generation compared to existing methods. Our code and pre-processed data are available at <a href="https://github.com/nhattruongpham/mol-lang-bridge/tree/mol2lang" rel="external nofollow noopener" target="_blank">https://github.com/nhattruongpham/mol-lang-bridge/tree/mol2lang/</a>.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@inproceedings</span><span class="p">{</span><span class="nl">tran2024mollangvlm</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Mol2Lang-{VLM}: Vision- and Text-Guided Generative Pre-trained Language Models for Advancing Molecule Captioning through Multimodal Fusion}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Tran, Duong Thanh and Pham, Nhat Truong and Nguyen, Nguyen Doan Hieu and Manavalan, Balachandran}</span><span class="p">,</span>
  <span class="na">booktitle</span> <span class="p">=</span> <span class="s">{Proceedings of the 1st Workshop on Language + Molecules L+M 2024}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{97--102}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.18653/v1/2024.langmol-1.12}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#215d42"> <a href="https://language-plus-molecules.github.io/" rel="external nofollow noopener" target="_blank">LPM@ACL</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/Lang2Mol-Diff.png" sizes="200px"></source> <img src="/assets/img/publication_preview/Lang2Mol-Diff.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="Lang2Mol-Diff.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="nguyen2024langmoldiff" class="col-sm-8"> <div class="title"> <b><i>Lang2Mol-Diff:</i></b> A Diffusion-Based Generative Model for Language-to-Molecule Translation Leveraging SELFIES Molecular String Representation</div> <div class="author"> <a href="https://scholar.google.com/citations?hl=en&amp;user=-aEoZCgAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Nguyen Doan Hieu Nguyen<sup>†</sup></a> , <em><b style="color: #FFA500;">Nhat Truong Pham<sup>†</sup></b></em> , <a href="https://scholar.google.com/citations?hl=en&amp;user=kz_chQ4AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Duong Thanh Tran</a> , and <a href="https://scholar.google.com/citations?hl=en&amp;user=0vkenbwAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Balachandran Manavalan</a> </div> <div class="periodical"> <em>In <i><a href="https://language-plus-molecules.github.io/" style="color: #00ab37; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Proceedings of the 1st Workshop on Language + Molecules (L+M 2024)</a></i></em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://aclanthology.org/2024.langmol-1.15/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://aclanthology.org/2024.langmol-1.15.pdf" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">PDF</a> <a href="https://github.com/nhattruongpham/mol-lang-bridge/tree/lang2mol" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> </div> <div class="badges"> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:isU91gLudPYC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-0-4285F4?logo=googlescholar&amp;labelColor=beige" alt="0 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Generating <i>de novo</i> molecules from textual descriptions is challenging due to potential issues with molecule validity in SMILES representation and limitations of autoregressive models. This work introduces Lang2Mol-Diff, a diffusion-based language-to-molecule generative model using the SELFIES representation. Specifically, Lang2Mol-Diff leverages the strengths of two state-of-the-art molecular generative models: BioT5 and TGM-DLM. By employing BioT5 to tokenize the SELFIES representation, Lang2Mol-Diff addresses the validity issues associated with SMILES strings. Additionally, it incorporates a text diffusion mechanism from TGM-DLM to overcome the limitations of autoregressive models in this domain. To the best of our knowledge, this is the first study to leverage the diffusion mechanism for text-based <i>de novo</i> molecule generation using the SELFIES molecular string representation. Performance evaluation on the L+M-24 benchmark dataset shows that Lang2Mol-Diff outperforms all existing methods for molecule generation in terms of validity. Our code and pre-processed data are available at <a href="https://github.com/nhattruongpham/mol-lang-bridge/tree/lang2mol" rel="external nofollow noopener" target="_blank">https://github.com/nhattruongpham/mol-lang-bridge/tree/lang2mol/</a>.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@inproceedings</span><span class="p">{</span><span class="nl">nguyen2024langmoldiff</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Lang2Mol-Diff: A Diffusion-Based Generative Model for Language-to-Molecule Translation Leveraging SELFIES Molecular String Representation}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Nguyen, Nguyen Doan Hieu and Pham, Nhat Truong and Tran, Duong Thanh and Manavalan, Balachandran}</span><span class="p">,</span>
  <span class="na">booktitle</span> <span class="p">=</span> <span class="s">{Proceedings of the 1st Workshop on Language + Molecules L+M 2024}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{128--134}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.18653/v1/2024.langmol-1.15}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://www.sciencedirect.com/journal/computers-in-biology-and-medicine" rel="external nofollow noopener" target="_blank">CIBM</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/HOTGpred.png" sizes="200px"></source> <img src="/assets/img/publication_preview/HOTGpred.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="HOTGpred.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="pham2024hotgpred" class="col-sm-8"> <div class="title"> <b><i>HOTGpred:</i></b> Enhancing human O-linked threonine glycosylation prediction using integrated pretrained protein language model-based features and multi-stage feature selection approach</div> <div class="author"> <em><b style="color: #FFA500;">Nhat Truong Pham<sup>†</sup></b></em> , Ying Zhang<sup>†</sup> , <a href="https://scholar.google.com/citations?hl=en&amp;user=5WP7pOUAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Rajan Rakkiyappan</a> , and <a href="https://scholar.google.com/citations?hl=en&amp;user=0vkenbwAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Balachandran Manavalan</a> </div> <div class="periodical"> <em><i><a href="https://www.sciencedirect.com/journal/computers-in-biology-and-medicine" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Computers in Biology and Medicine</a></i></em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/abs/pii/S0010482524009442" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://balalab-skku.org/HOTGpred/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Website</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.compbiomed.2024.108859" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:2ywjKiB__4kC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-4-4285F4?logo=googlescholar&amp;labelColor=beige" alt="4 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>O-linked glycosylation is a complex post-translational modification (PTM) in human proteins that plays a critical role in regulating various cellular metabolic and signaling pathways. In contrast to N-linked glycosylation, O-linked glycosylation lacks specific sequence features and maintains an unstable core structure. Identifying O-linked threonine glycosylation sites (OTGs) remains challenging, requiring extensive experimental tests. While bioinformatics tools have emerged for predicting OTGs, their reliance on limited conventional features and absence of well-defined feature selection strategies limit their effectiveness. To address these limitations, we introduced HOTGpred (Human O-linked Threonine Glycosylation predictor), employing a multi-stage feature selection process to identify the optimal feature set for accurately identifying OTGs. Initially, we assessed 25 different feature sets derived from various pretrained protein language model (PLM)-based embeddings and conventional feature descriptors using nine classifiers. Subsequently, we integrated the top five embeddings linearly and determined the most effective scoring function for ranking hybrid features, identifying the optimal feature set through a process of sequential forward search. Among the classifiers, the extreme gradient boosting (XGBT)-based model, using the optimal feature set (HOTGpred), achieved 92.03% accuracy on the training dataset and 88.25% on the balanced independent dataset. Notably, HOTGpred significantly outperformed the current state-of-the-art methods on both the balanced and imbalanced independent datasets, demonstrating its superior prediction capabilities. Additionally, SHapley Additive exPlanations (SHAP) and ablation analyses were conducted to identify the features contributing most significantly to HOTGpred. Finally, we developed an easy-to-navigate web server, accessible at <a href="https://balalab-skku.org/HOTGpred/" rel="external nofollow noopener" target="_blank">https://balalab-skku.org/HOTGpred/</a>, to support glycobiologists in their research on glycosylation structure and function.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">pham2024hotgpred</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{HOTGpred: Enhancing human O-linked threonine glycosylation prediction using integrated pretrained protein language model-based features and multi-stage feature selection approach}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Pham, Nhat Truong and Zhang, Ying and Rakkiyappan, Rajan and Manavalan, Balachandran}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Computers in Biology and Medicine}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{179}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{108859}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.compbiomed.2024.108859}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://www.sciencedirect.com/journal/journal-of-molecular-biology" rel="external nofollow noopener" target="_blank">JMB</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/mACPpred2.png" sizes="200px"></source> <img src="/assets/img/publication_preview/mACPpred2.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="mACPpred2.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="sangaraju2024macppred" class="col-sm-8"> <div class="title"> <b><i>mACPpred 2.0:</i></b> Stacked Deep Learning for Anticancer Peptide Prediction with Integrated Spatial and Probabilistic Feature Representations</div> <div class="author"> Vinoth Kumar Sangaraju<sup>†</sup> , <em><b style="color: #FFA500;">Nhat Truong Pham<sup>†</sup></b></em> , <a href="https://scholar.google.com/citations?hl=en&amp;user=0EAV03MAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Leyi Wei</a> , Xue Yu , and <a href="https://scholar.google.com/citations?hl=en&amp;user=0vkenbwAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Balachandran Manavalan</a> </div> <div class="periodical"> <em><i><a href="https://www.sciencedirect.com/journal/journal-of-molecular-biology" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Journal of Molecular Biology</a></i></em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/abs/pii/S0022283624002894" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://github.com/nhattruongpham/mACPpred2/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> <a href="https://balalab-skku.org/mACPpred2/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Website</a> <a href="https://doi.org/10.5281/zenodo.11350064" rel="external nofollow noopener" target="_blank"> <img src="https://zenodo.org/badge/doi/10.5281/zenodo.11350064.svg" alt="DOI"> </a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.jmb.2024.168687" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:1tZ8xJnm2c8C" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-9-4285F4?logo=googlescholar&amp;labelColor=beige" alt="9 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Anticancer peptides (ACPs), naturally occurring molecules with remarkable potential to target and kill cancer cells. However, identifying ACPs based solely from their primary amino acid sequences remains a major hurdle in immunoinformatics. In the past, several web-based machine learning (ML) tools have been proposed to assist researchers in identifying potential ACPs for further testing. Notably, our meta- approach method, mACPpred, introduced in 2019, has significantly advanced the field of ACP research. Given the exponential growth in the number of characterized ACPs, there is now a pressing need to create an updated version of mACPpred. To develop mACPpred 2.0, we constructed an up-to-date benchmarking dataset by integrating all publicly available ACP datasets. We employed a large-scale of feature descriptors, encompassing both conventional feature descriptors and advanced pre-trained natural language processing (NLP)-based embeddings. We evaluated their ability to discriminate between ACPs and non-ACPs using eleven different classifiers. Subsequently, we employed a stacked deep learning (SDL) approach, incorporating 1D convolutional neural network (1D CNN) blocks and hybrid features. These features included the top seven performing NLP-based features and 90 probabilistic features, allowing us to identify hidden patterns within these diverse features and improve the accuracy of our ACP prediction model. This is the first study to integrate spatial and probabilistic feature representations for predicting ACPs. Rigorous cross-validation and independent tests conclusively demonstrated that mACPpred 2.0 not only surpassed its predecessor (mACPpred) but also outperformed the existing state-of-the-art predictors, highlighting the importance of advanced feature representation capabilities attained through SDL. To facilitate widespread use and accessibility, we have developed a user-friendly for mACPpred 2.0, available at <a href="https://balalab-skku.org/mACPpred2/" rel="external nofollow noopener" target="_blank">https://balalab-skku.org/mACPpred2/</a>.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">sangaraju2024macppred</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{mACPpred 2.0: Stacked Deep Learning for Anticancer Peptide Prediction with Integrated Spatial and Probabilistic Feature Representations}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Sangaraju, Vinoth Kumar and Pham, Nhat Truong and Wei, Leyi and Yu, Xue and Manavalan, Balachandran}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Journal of Molecular Biology}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{436}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{17}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{168687}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.jmb.2024.168687}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://www.sciencedirect.com/journal/international-journal-of-biological-macromolecules" rel="external nofollow noopener" target="_blank">IJBIOMAC</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/SEP-AlgPro.png" sizes="200px"></source> <img src="/assets/img/publication_preview/SEP-AlgPro.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="SEP-AlgPro.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="basith2024sep" class="col-sm-8"> <div class="title"> <b><i>SEP-AlgPro:</i></b> An efficient allergen prediction tool utilizing traditional machine learning and deep learning techniques with protein language model features</div> <div class="author"> <a href="https://scholar.google.com/citations?hl=en&amp;user=2eqv0J4AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Shaherin Basith</a> , <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , <a href="https://scholar.google.com/citations?hl=en&amp;user=0vkenbwAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Balachandran Manavalan</a> , and Gwang Lee </div> <div class="periodical"> <em><i><a href="https://www.sciencedirect.com/journal/international-journal-of-biological-macromolecules" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">International Journal of Biological Macromolecules</a></i></em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/abs/pii/S014181302403890X" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://balalab-skku.org/SEP-AlgPro/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Website</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.ijbiomac.2024.133085" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:YlPif8NxrbYC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-4-4285F4?logo=googlescholar&amp;labelColor=beige" alt="4 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Allergy is a hypersensitive condition in which individuals develop objective symptoms when exposed to harmless substances at a dose that would cause no harm to a “normal” person. Most current computational methods for allergen identification rely on homology or conventional machine learning using limited set of feature descriptors or validation on specific datasets, making them inefficient and inaccurate. Here, we propose SEP-AlgPro for the accurate identification of allergen protein from sequence information. We analyzed 10 conventional protein-based features and 14 different features derived from protein language models to gauge their effectiveness in differentiating allergens from non-allergens using 15 different classifiers. However, the final optimized model employs top 10 feature descriptors with top seven machine learning classifiers. Results show that the features derived from protein language models exhibit superior discriminative capabilities compared to traditional feature sets. This enabled us to select the most discriminatory baseline models, whose predicted outputs were aggregated and used as input to a deep neural network for the final allergen prediction. Extensive case studies showed that SEP-AlgPro outperforms state-of-the-art predictors in accurately identifying allergens. A user-friendly web server was developed and made freely available at <a href="https://balalab-skku.org/SEP-AlgPro/" rel="external nofollow noopener" target="_blank">https://balalab-skku.org/SEP-AlgPro/</a>, making it a powerful tool for identifying potential allergens.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">basith2024sep</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{SEP-AlgPro: An efficient allergen prediction tool utilizing traditional machine learning and deep learning techniques with protein language model features}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Basith, Shaherin and Pham, Nhat Truong and Manavalan, Balachandran and Lee, Gwang}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{International Journal of Biological Macromolecules}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{273}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{133085}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.ijbiomac.2024.133085}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://link.springer.com/journal/10661" rel="external nofollow noopener" target="_blank">EMA</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/DroughtLD.png" sizes="200px"></source> <img src="/assets/img/publication_preview/DroughtLD.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="DroughtLD.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="thanh2024predicting" class="col-sm-8"> <div class="title">Predicting drought stress under climate change in the Southern Central Highlands of Vietnam</div> <div class="author"> <a href="https://scholar.google.com/citations?hl=en&amp;user=PYCmxMQAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Phong Nguyen Thanh</a> , Xuan Ai Tien Van , Au Nguyen Hai , Chinh Le Cong , <a href="https://scholar.google.com/citations?hl=en&amp;user=a-dNtokAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Alexandre S Gagnon</a> , <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , <a href="https://scholar.google.com/citations?hl=en&amp;user=4MfplgUAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Duong Tran Anh</a> , and Vuong Nguyen Dinh </div> <div class="periodical"> <em><i><a href="https://link.springer.com/journal/10661" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Environmental Monitoring and Assessment</a></i></em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://link.springer.com/article/10.1007/s10661-024-12798-6" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1007/s10661-024-12798-6" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:ghEM2AJqZyQC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-1-4285F4?logo=googlescholar&amp;labelColor=beige" alt="1 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>In the Southern Central Highlands of Vietnam, droughts occur more frequently, causing significant damage and impacting the region’s socio-economic development. During the dry season, rivers, streams, and reservoirs often face limited water availability, exacerbated in recent years by increasing drought severity. Recognizing the escalating severity of droughts, the study offers a novel contribution by conducting a comprehensive analysis of surface water resource distribution in Lam Dong province, focusing on assessing water demand for agricultural production, a crucial factor in ensuring sustainable crop growth. Two scenarios, Current-2020 (SC1) and Climate Change-2025 (SC2), are simulated, with SC2 based on climate change and sea level rise scenarios provided by the Ministry of Natural Resources and Environment (MONRE). These scenarios are integrated into the MIKE-NAM and MIKE-HYDRO basin models, allowing for a thorough assessment of the water balance of Lam Dong province. Furthermore, the study utilizes the Keetch–Byram Drought Index (KBDI) to measure drought severity, revealing prevalent dry and moderately droughty conditions in highland districts with precipitation ranging from 50% to 85%. Severe drought conditions emerge at 95% precipitation levels, indicating an increased frequency and geographic scope of severe droughts. Additionally, the study highlights that under abnormally dry conditions, water demand for the winter-spring crop is consistently met at 100%, decreasing to 85%, 80%, and less than 75% for moderate, severe, and extreme droughts, respectively. These findings offer insights into future drought conditions in the Lam Dong province and their potential impact on irrigation capacity, crucial for adaptation strategies.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">thanh2024predicting</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Predicting drought stress under climate change in the Southern Central Highlands of Vietnam}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Thanh, Phong Nguyen and Van, Thinh Le Thi, Xuan Ai Tien and Hai, Au Nguyen and Cong, Chinh Le and Gagnon, Alexandre S and Pham, Nhat Truong and Anh, Duong Tran and Dinh, Vuong Nguyen}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Environmental Monitoring and Assessment}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{196}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{7}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{636}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Springer}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/s10661-024-12798-6}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://journals.plos.org/plosone/" rel="external nofollow noopener" target="_blank">PONE</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/Meta-2OM.png" sizes="200px"></source> <img src="/assets/img/publication_preview/Meta-2OM.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="Meta-2OM.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="harun2024meta" class="col-sm-8"> <div class="title"> <b><i>Meta-2OM:</i></b> A multi-classifier meta-model for the accurate prediction of RNA 2’-O-methylation sites in human RNA</div> <div class="author"> Md Harun-Or-Roshid , <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , <a href="https://scholar.google.com/citations?hl=en&amp;user=0vkenbwAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Balachandran Manavalan</a> , and Hiroyuki Kurata </div> <div class="periodical"> <em><i><a href="https://journals.plos.org/plosone/" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">PLoS ONE</a></i></em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0305406" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/Meta-2OM.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a> <a href="https://github.com/kuratahiroyuki/Meta-2OM" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> <a href="http://kurata35.bio.kyutech.ac.jp/Meta-2OM/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Website</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pone.0305406" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:0aBXIfxlw9sC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-3-4285F4?logo=googlescholar&amp;labelColor=beige" alt="3 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>2’-O-methylation (2-OM or Nm) is a widespread RNA modification observed in various RNA types like tRNA, mRNA, rRNA, miRNA, piRNA, and snRNA, which plays a crucial role in several biological functional mechanisms and innate immunity. To comprehend its modification mechanisms and potential epigenetic regulation, it is necessary to accurately identify 2-OM sites. However, biological experiments can be tedious, time-consuming, and expensive. Furthermore, currently available computational methods face challenges due to inadequate datasets and limited classification capabilities. To address these challenges, we proposed Meta-2OM, a cutting-edge predictor that can accurately identify 2-OM sites in human RNA. In brief, we applied a meta-learning approach that considered eight conventional machine learning algorithms, including tree-based classifiers and decision boundary-based classifiers, and eighteen different feature encoding algorithms that cover physicochemical, compositional, position-specific and natural language processing information. The predicted probabilities of 2-OM sites from the baseline models are then combined and trained using logistic regression to generate the final prediction. Consequently, Meta-2OM achieved excellent performance in both 5-fold cross-validation training and independent testing, outperforming all existing state-of-the-art methods. Specifically, on the independent test set, Meta-2OM achieved an overall accuracy of 0.870, sensitivity of 0.836, specificity of 0.904, and Matthews correlation coefficient of 0.743. To facilitate its use, a user-friendly web server and standalone program have been developed and freely available at <a href="http://kurata35.bio.kyutech.ac.jp/Meta-2OM/" rel="external nofollow noopener" target="_blank">http://kurata35.bio.kyutech.ac.jp/Meta-2OM/</a> and <a href="https://github.com/kuratahiroyuki/Meta-2OM/" rel="external nofollow noopener" target="_blank">https://github.com/kuratahiroyuki/Meta-2OM/</a>.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">harun2024meta</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Meta-2OM: A multi-classifier meta-model for the accurate prediction of RNA 2'-O-methylation sites in human RNA}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Harun-Or-Roshid, Md and Pham, Nhat Truong and Manavalan, Balachandran and Kurata, Hiroyuki}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{PLoS ONE}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{19}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{6}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{e0305406}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Public Library of Science San Francisco, CA USA}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1371/journal.pone.0305406}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://www.cell.com/molecular-therapy-family/nucleic-acids/home" rel="external nofollow noopener" target="_blank">MTNA</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/ac4C-AFL.png" sizes="200px"></source> <img src="/assets/img/publication_preview/ac4C-AFL.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="ac4C-AFL.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="pham2024ac4c" class="col-sm-8"> <div class="title"> <b><i>ac4C-AFL:</i></b> A high-precision identification of human mRNA N4-acetylcytidine sites based on adaptive feature representation learning</div> <div class="author"> <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , Annie Terrina Terrance , <a href="https://scholar.google.com/citations?hl=en&amp;user=DkVnJ-wAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Young-Jun Jeon</a> , <a href="https://scholar.google.com/citations?hl=en&amp;user=5WP7pOUAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Rajan Rakkiyappan</a> , and <a href="https://scholar.google.com/citations?hl=en&amp;user=0vkenbwAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Balachandran Manavalan</a> </div> <div class="periodical"> <em><i><a href="https://www.cell.com/molecular-therapy-family/nucleic-acids/home" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Molecular Therapy-Nucleic Acids</a></i></em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.cell.com/molecular-therapy-family/nucleic-acids/fulltext/S2162-2531(24)00079-9" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/ac4C-AFL.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a> <a href="https://balalab-skku.org/ac4C-AFL/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Website</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.omtn.2024.102192" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:27LrP4qxOz0C" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-10-4285F4?logo=googlescholar&amp;labelColor=beige" alt="10 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>RNA N4-acetylcytidine (ac4C) is a highly conserved RNA modification that plays a crucial role in controlling mRNA stability, processing, and translation. Consequently, accurate identification of ac4C sites across the genome is critical for understanding gene expression regulation mechanisms. In this study, we have developed ac4C-AFL, a bioinformatics tool that precisely identifies ac4C sites from primary RNA sequences. In ac4C-AFL, we identified the optimal sequence length for model building and implemented an adaptive feature representation strategy that is capable of extracting the most representative features from RNA. To identify the most relevant features, we proposed a novel ensemble feature importance scoring strategy to rank features effectively. We then used this information to conduct the sequential forward search, which individually determine the optimal feature set from the 16 sequence-derived feature descriptors. Utilizing these optimal feature descriptors, we constructed 176 baseline models using 11 popular classifiers. The most efficient baseline models were identified using the two-step feature selection approach, whose predicted scores were integrated and trained with the appropriate classifier to develop the final prediction model. Our rigorous cross-validations and independent tests demonstrate that ac4C-AFL surpasses contemporary tools in predicting ac4C sites. Moreover, we have developed a publicly accessible web server at <a href="https://balalab-skku.org/ac4C-AFL/" rel="external nofollow noopener" target="_blank">https://balalab-skku.org/ac4C-AFL/</a>.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">pham2024ac4c</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{ac4C-AFL: A high-precision identification of human mRNA N4-acetylcytidine sites based on adaptive feature representation learning}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Pham, Nhat Truong and Terrance, Annie Terrina and Jeon, Young-Jun and Rakkiyappan, Rajan and Manavalan, Balachandran}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Molecular Therapy-Nucleic Acids}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{35}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{102192}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.omtn.2024.102192}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#215d42"> <a href="https://www.iciit.org/index.html" rel="external nofollow noopener" target="_blank">ICIIT</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/ICIIT24.png" sizes="200px"></source> <img src="/assets/img/publication_preview/ICIIT24.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="ICIIT24.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="phan2024innovative" class="col-sm-8"> <div class="title">Innovative Multi-Modal Control for Surveillance Spider Robot: An Integration of Voice and Hand Gesture Recognition</div> <div class="author"> Dang Khoa Phan , <a href="https://scholar.google.com/citations?hl=en&amp;user=NKbwDD8AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Phuong-Nam Tran</a> , <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , Tra Huong Thi Le , and <a href="https://scholar.google.com/citations?hl=en&amp;user=2UKP440AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Duc Ngoc Minh Dang</a> </div> <div class="periodical"> <em>In <i><a href="https://www.iciit.org/index.html" style="color: #00ab37; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Proceedings of the 2024 9th International Conference on Intelligent Information Technology</a></i></em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://dl.acm.org/doi/10.1145/3654522.3654544" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1145/3654522.3654544" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:eAlLMO4JVmQC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-0-4285F4?logo=googlescholar&amp;labelColor=beige" alt="0 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>The spider robot is designed to take on challenging tasks in hazardous conditions. It can move across challenging terrain like walls and rough surfaces, and effectively find lost objects. In this paper, an innovative multi-modal control approach was developed for the Surveillance Spider Robot (SSR) application, integrating voice recognition and hand gesture recognition as control commands. SSR, a six-legged robot, was designed using a Raspberry Pi 4B embedded device, Arduino Uno kit, RC Servo motors (MG996R), 18650 batteries, mini USB microphone (MI-350), Pi camera V1 (OV5647) and PWM generator (PCA9685). The robot can be controlled through voice or hand gesture recognition captured via camera and microphone. SSR is capable of performing ten specific tasks based on these control commands, including forward movement, backward movement, left turns, right turns, complete turns, movements with higher or lower centers of gravity, slow movement, body-hopping, and stopping. The performance evaluation of voice and hand gesture recognition suggested that SSR can be used in real-world applications with an accuracy that exceeds 90% for the ten specific tasks.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@inproceedings</span><span class="p">{</span><span class="nl">phan2024innovative</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Innovative Multi-Modal Control for Surveillance Spider Robot: An Integration of Voice and Hand Gesture Recognition}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Phan, Dang Khoa and Tran, Phuong-Nam and Pham, Nhat Truong and Le, Tra Huong Thi and Dang, Duc Ngoc Minh}</span><span class="p">,</span>
  <span class="na">booktitle</span> <span class="p">=</span> <span class="s">{Proceedings of the 2024 9th International Conference on Intelligent Information Technology}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{141--148}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1145/3654522.3654544}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#215d42"> <a href="https://www.iciit.org/index.html" rel="external nofollow noopener" target="_blank">ICIIT</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/ICIIT24.png" sizes="200px"></source> <img src="/assets/img/publication_preview/ICIIT24.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="ICIIT24.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="phan2024deep" class="col-sm-8"> <div class="title">Deep Learning-Based Automated Cashier System for Bakeries</div> <div class="author"> Nam Van Hai Phan , Tha Thanh Le , Tuan Phu Phan , <a href="https://scholar.google.com/citations?hl=en&amp;user=l-bJWlcAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Thu Thuy Le</a> , <a href="https://scholar.google.com/citations?hl=en&amp;user=NKbwDD8AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Phuong-Nam Tran</a> , <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , and <a href="https://scholar.google.com/citations?hl=en&amp;user=2UKP440AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Duc Ngoc Minh Dang</a> </div> <div class="periodical"> <em>In <i><a href="https://www.iciit.org/index.html" style="color: #00ab37; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Proceedings of the 2024 9th International Conference on Intelligent Information Technology</a></i></em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://dl.acm.org/doi/10.1145/3654522.3654538" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1145/3654522.3654538" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:RJOyoaXV5v8C" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-0-4285F4?logo=googlescholar&amp;labelColor=beige" alt="0 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>The application of image recognition in the bakery business has paved the way for automatic payment systems, a significant advancement in the field of computer vision. This article delves into an exploration of advanced image recognition models to meticulously assess their effectiveness, speed, and suitability for seamless integration into specialized automatic payment systems tailored for bakeries. Specifically, YOLOX, YOLOv8, Faster R-CNN, and RetinaNet, each with different versions and backbones, are considered for evaluation based on their speed and performance. Notably, this study introduces a streamlined process for rapidly creating custom datasets for object detection research and evaluates models across these datasets. The insights and analyses derived from this study provide valuable perspectives for optimizing processes and enhancing the overall performance of automatic payment systems within bakeries.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@inproceedings</span><span class="p">{</span><span class="nl">phan2024deep</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Deep Learning-Based Automated Cashier System for Bakeries}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Phan, Nam Van Hai and Le, Tha Thanh and Phan, Tuan Phu and Le, Thu Thuy and Tran, Phuong-Nam and Pham, Nhat Truong and Dang, Duc Ngoc Minh}</span><span class="p">,</span>
  <span class="na">booktitle</span> <span class="p">=</span> <span class="s">{Proceedings of the 2024 9th International Conference on Intelligent Information Technology}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{94--100}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1145/3654522.3654538}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://academic.oup.com/bib" rel="external nofollow noopener" target="_blank">BiB</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/H2Opred.png" sizes="200px"></source> <img src="/assets/img/publication_preview/H2Opred.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="H2Opred.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="pham2024h2opred" class="col-sm-8"> <div class="title"> <b><i>H2Opred:</i></b> a robust and efficient hybrid deep learning model for predicting 2’-O-methylation sites in human RNA</div> <div class="author"> <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , <a href="https://scholar.google.com/citations?hl=en&amp;user=5WP7pOUAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Rajan Rakkiyappan</a> , <a href="https://scholar.google.com/citations?hl=en&amp;user=_YZClBgAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Jongsun Park</a> , <a href="https://scholar.google.com/citations?hl=en&amp;user=IRshBmMAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Adeel Malik</a> , and <a href="https://scholar.google.com/citations?hl=en&amp;user=0vkenbwAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Balachandran Manavalan</a> </div> <div class="periodical"> <em><i><a href="https://academic.oup.com/bib" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Briefings in Bioinformatics</a></i></em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://academic.oup.com/bib/article/25/1/bbad476/7510980" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/H2Opred.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a> <a href="https://balalab-skku.org/H2Opred/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Website</a> <a href="https://www.ibric.org/s.do?CWXEPvOoln" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Media</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1093/bib/bbad476" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:QsaTk4IG4EwC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-23-4285F4?logo=googlescholar&amp;labelColor=beige" alt="23 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>2’-O-methylation (2OM) is the most common post-transcriptional modification of RNA. It plays a crucial role in RNA splicing, RNA stability and innate immunity. Despite advances in high-throughput detection, the chemical stability of 2OM makes it difficult to detect and map in messenger RNA. Therefore, bioinformatics tools have been developed using machine learning (ML) algorithms to identify 2OM sites. These tools have made significant progress, but their performances remain unsatisfactory and need further improvement. In this study, we introduced H2Opred, a novel hybrid deep learning (HDL) model for accurately identifying 2OM sites in human RNA. Notably, this is the first application of HDL in developing four nucleotide-specific models [adenine (A2OM), cytosine (C2OM), guanine (G2OM) and uracil (U2OM)] as well as a generic model (N2OM). H2Opred incorporated both stacked 1D convolutional neural network (1D-CNN) blocks and stacked attention-based bidirectional gated recurrent unit (Bi-GRU-Att) blocks. 1D-CNN blocks learned effective feature representations from 14 conventional descriptors, while Bi-GRU-Att blocks learned feature representations from five natural language processing-based embeddings extracted from RNA sequences. H2Opred integrated these feature representations to make the final prediction. Rigorous cross-validation analysis demonstrated that H2Opred consistently outperforms conventional ML-based single-feature models on five different datasets. Moreover, the generic model of H2Opred demonstrated a remarkable performance on both training and testing datasets, significantly outperforming the existing predictor and other four nucleotide-specific H2Opred models. To enhance accessibility and usability, we have deployed a user-friendly web server for H2Opred, accessible at <a href="https://balalab-skku.org/H2Opred/" rel="external nofollow noopener" target="_blank">https://balalab-skku.org/H2Opred/</a>. This platform will serve as an invaluable tool for accurately predicting 2OM sites within human RNA, thereby facilitating broader applications in relevant research endeavors.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">pham2024h2opred</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{H2Opred: a robust and efficient hybrid deep learning model for predicting 2'-O-methylation sites in human RNA}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Pham, Nhat Truong and Rakkiyappan, Rajan and Park, Jongsun and Malik, Adeel and Manavalan, Balachandran}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Briefings in Bioinformatics}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{25}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{bbad476}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{OxfOxford University Pressord}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1093/bib/bbad476}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://ietresearch.onlinelibrary.wiley.com/journal/17518695" rel="external nofollow noopener" target="_blank">IET GTD</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/HBA-dHoSMO.png" sizes="200px"></source> <img src="/assets/img/publication_preview/HBA-dHoSMO.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="HBA-dHoSMO.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="tran2024enhanced" class="col-sm-8"> <div class="title">Enhanced sliding mode controller design via meta-heuristic algorithm for robust and stable load frequency control in multi-area power systems</div> <div class="author"> <a href="https://scholar.google.com/citations?hl=en&amp;user=xrvf7L4AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Anh-Tuan Tran</a> , Minh Phuc Duong , <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , and <a href="https://scholar.google.com/citations?hl=en&amp;user=xl6iO0QAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Jae Woong Shim</a> </div> <div class="periodical"> <em><i><a href="https://ietresearch.onlinelibrary.wiley.com/journal/17518695" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">IET Generation, Transmission &amp; Distribution</a></i></em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://ietresearch.onlinelibrary.wiley.com/doi/full/10.1049/gtd2.13077" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/HBA-dHoSMO.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1049/gtd2.13077" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:aIdbFUkbNIkC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-10-4285F4?logo=googlescholar&amp;labelColor=beige" alt="10 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>This article introduces a novel approach named HBA-dHoSMO, which combines a continuous decentralized higher-order sliding mode controller-based observer (dHoSMO) with the honey badger algorithm (HBA), specifically designed for load frequency control (LFC) in multi-area power systems (MAPSs). Traditional sliding mode controllers (SMCs) employed in LFC of MAPSs often face challenges related to chattering and oscillations, leading to decreased robustness and stability. Additionally, tuning the parameters for these SMC designs to achieve optimal performance in MAPSs can be challenging. The HBA-dHoSMO is proposed to address the issues of chattering and oscillations, while the optimal parameters for SMC design are obtained using HBA. The stability analysis of the entire system is conducted using linear matrix inequality and the Lyapunov stability theory, affirming the reliability and feasibility of the approach. A comprehensive set of case studies is performed under various configurations and conditions. Additionally, particle swarm optimization and tuna swarm optimization, in conjunction with SMC-based and proportional-integral-derivative controllers, are examined for performance comparison. Simulation results demonstrate the superior performance of the proposed controller across all case studies. This is evidenced by the lowest integral time absolute error values recorded as 0.0133, 0.0006, and 0.0167 for single-, two-, and three-area power systems, respectively.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">tran2024enhanced</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Enhanced sliding mode controller design via meta-heuristic algorithm for robust and stable load frequency control in multi-area power systems}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Tran, Anh-Tuan and Duong, Minh Phuc and Pham, Nhat Truong and Shim, Jae Woong}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{IET Generation, Transmission &amp; Distribution}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{18}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{460--478}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Wiley}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1049/gtd2.13077}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://academic.oup.com/bib" rel="external nofollow noopener" target="_blank">BiB</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/MeL-STPhos.png" sizes="200px"></source> <img src="/assets/img/publication_preview/MeL-STPhos.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="MeL-STPhos.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="pham2024advancing" class="col-sm-8"> <div class="title">Advancing the accuracy of SARS-CoV-2 phosphorylation site detection via meta-learning approach</div> <div class="author"> <em><b style="color: #FFA500;">Nhat Truong Pham<sup>†</sup></b></em> , Le Thi Phan<sup>†</sup> , Jimin Seo , Yeonwoo Kim , <a href="https://scholar.google.com/citations?hl=en&amp;user=sBSJfo4AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Minkyung Song</a> , Sukchan Lee , <a href="https://scholar.google.com/citations?hl=en&amp;user=DkVnJ-wAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Young-Jun Jeon</a> , and <a href="https://scholar.google.com/citations?hl=en&amp;user=0vkenbwAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Balachandran Manavalan</a> </div> <div class="periodical"> <em><i><a href="https://academic.oup.com/bib" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Briefings in Bioinformatics</a></i></em>, 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://academic.oup.com/bib/article/25/1/bbad433/7459584" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/MeL-STPhos.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a> <a href="https://balalab-skku.org/MeL-STPhos/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Website</a> <a href="https://www.ibric.org/s.do?nLdsctWFBW" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Media</a> <a href="https://www.ibric.org/s.do?ovlRZSpKQk" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Interview</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1093/bib/bbad433" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:k_7cPK9k7w8C" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-20-4285F4?logo=googlescholar&amp;labelColor=beige" alt="20 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>The worldwide appearance of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) has generated significant concern and posed a considerable challenge to global health. Phosphorylation is a common post-translational modification that affects many vital cellular functions and is closely associated with SARS-CoV-2 infection. Precise identification of phosphorylation sites could provide more in-depth insight into the processes underlying SARS-CoV-2 infection and help alleviate the continuing coronavirus disease 2019 (COVID-19) crisis. Currently, available computational tools for predicting these sites lack accuracy and effectiveness. In this study, we designed an innovative meta-learning model, Meta-Learning for Serine/Threonine Phosphorylation (MeL-STPhos), to precisely identify protein phosphorylation sites. We initially performed a comprehensive assessment of 29 unique sequence-derived features, establishing prediction models for each using 14 renowned machine learning methods, ranging from traditional classifiers to advanced deep learning algorithms. We then selected the most effective model for each feature by integrating the predicted values. Rigorous feature selection strategies were employed to identify the optimal base models and classifier(s) for each cell-specific dataset. To the best of our knowledge, this is the first study to report two cell-specific models and a generic model for phosphorylation site prediction by utilizing an extensive range of sequence-derived features and machine learning algorithms. Extensive cross-validation and independent testing revealed that MeL-STPhos surpasses existing state-of-the-art tools for phosphorylation site prediction. We also developed a publicly accessible platform at <a href="https://balalab-skku.org/MeL-STPhos/" rel="external nofollow noopener" target="_blank">https://balalab-skku.org/MeL-STPhos/</a>. We believe that MeL-STPhos will serve as a valuable tool for accelerating the discovery of serine/threonine phosphorylation sites and elucidating their role in post-translational regulation.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">pham2024advancing</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Advancing the accuracy of SARS-CoV-2 phosphorylation site detection via meta-learning approach}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Pham, Nhat Truong and Phan, Le Thi and Seo, Jimin and Kim, Yeonwoo and Song, Minkyung and Lee, Sukchan and Jeon, Young-Jun and Manavalan, Balachandran}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Briefings in Bioinformatics}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{25}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{bbad433}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2024}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{OxfOxford University Pressord}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1093/bib/bbad433}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> <h2 class="bibliography">2023</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#215d42"> <a href="https://ictc.org/" rel="external nofollow noopener" target="_blank">ICTC</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/ICTC23.png" sizes="200px"></source> <img src="/assets/img/publication_preview/ICTC23.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="ICTC23.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="tran2023comparative" class="col-sm-8"> <div class="title">Comparative analysis of multi-loss functions for enhanced multi-modal speech emotion recognition</div> <div class="author"> <a href="https://scholar.google.com/citations?hl=en&amp;user=NKbwDD8AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Phuong-Nam Tran</a> , Thuy-Duong Thi Vu , <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , Hanh Dang-Ngoc , and <a href="https://scholar.google.com/citations?hl=en&amp;user=2UKP440AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Duc Ngoc Minh Dang</a> </div> <div class="periodical"> <em>In <i><a href="https://ictc.org/" style="color: #00ab37; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">2023 14th International Conference on Information and Communication Technology Convergence (ICTC)</a></i></em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://ieeexplore.ieee.org/document/10392928" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1109/ICTC58733.2023.10392928" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:g5Ck-dwhA_QC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-2-4285F4?logo=googlescholar&amp;labelColor=beige" alt="2 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>In recent years, multi-modal analysis has gained significant prominence across domains such as audio/speech processing, natural language processing, and affective computing, with a particular focus on speech emotion recognition (SER). The integration of data from diverse sources, encompassing text, audio, and images, in conjunction with classifier algorithms has led to the realization of enhanced performance in SER tasks. Traditionally, the cross-entropy loss function has been employed for the classification problem. However, it is challenging to discriminate the feature representations among classes for multi-modal classification tasks. In this study, we focus on the impact of the loss functions on multi-modal SER rather than designing the model architecture. Mainly, we evaluate the performance of multi-modal SER with different loss functions, such as cross-entropy loss, center loss, contrastive-center loss, and their combinations. Based on extensive comparative analysis, it is proven that the combination of cross-entropy loss and contrastive-center loss achieves the best performance for multi-modal SER. This combination reaches the highest accuracy of 80.27% and the highest balanced accuracy of 81.44% on the IEMOCAP dataset.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@inproceedings</span><span class="p">{</span><span class="nl">tran2023comparative</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Comparative analysis of multi-loss functions for enhanced multi-modal speech emotion recognition}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Tran, Phuong-Nam and Vu, Thuy-Duong Thi and Pham, Nhat Truong and Dang-Ngoc, Hanh and Dang, Duc Ngoc Minh}</span><span class="p">,</span>
  <span class="na">booktitle</span> <span class="p">=</span> <span class="s">{2023 14th International Conference on Information and Communication Technology Convergence ICTC}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{425--429}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">organization</span> <span class="p">=</span> <span class="s">{IEEE}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1109/ICTC58733.2023.10392928}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#215d42"> <a href="https://soict.org/" rel="external nofollow noopener" target="_blank">SOICT</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/SER-Fuse.png" sizes="200px"></source> <img src="/assets/img/publication_preview/SER-Fuse.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="SER-Fuse.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="pham2023serfuse" class="col-sm-8"> <div class="title"> <b><i>SER-Fuse:</i></b> An Emotion Recognition Application Utilizing Multi-Modal, Multi-Lingual, and Multi-Feature Fusion</div> <div class="author"> <em><b style="color: #FFA500;">Nhat Truong Pham<sup>*</sup></b></em> , Le Thi Phan , <a href="https://scholar.google.com/citations?hl=en&amp;user=2UKP440AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Duc Ngoc Minh Dang</a> , and <a href="https://scholar.google.com/citations?hl=en&amp;user=0vkenbwAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Balachandran Manavalan</a> </div> <div class="periodical"> <em>In <i><a href="https://soict.org/" style="color: #00ab37; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Proceedings of the 12th International Symposium on Information and Communication Technology</a></i></em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://dl.acm.org/doi/10.1145/3628797.3628887" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://github.com/nhattruongpham/SER-Fuse" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1145/3628797.3628887" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:LXmCCkuhhTsC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-2-4285F4?logo=googlescholar&amp;labelColor=beige" alt="2 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Speech emotion recognition (SER) is a crucial aspect of affective computing and human-computer interaction, yet effectively identifying emotions in different speakers and languages remains challenging. This paper introduces SER-Fuse, a multi-modal SER application that is designed to address the complexities of multiple speakers and languages. Our approach leverages diverse audio/speech embeddings and text embeddings to extract optimal features for multi-modal SER. We subsequently employ multi-feature fusion to integrate embedding features across modalities and languages. Experimental results archived on the English-Chinese emotional speech (ECES) dataset reveal that SER-Fuse attains competitive performance in the multi-lingual approach compared to the single-lingual approaches. Furthermore, we provide the implementation of SER-Fuse for download at <a href="https://github.com/nhattruongpham/SER-Fuse/" rel="external nofollow noopener" target="_blank">https://github.com/nhattruongpham/SER-Fuse/</a> to support reproducibility and local deployment.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@inproceedings</span><span class="p">{</span><span class="nl">pham2023serfuse</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{SER-Fuse: An Emotion Recognition Application Utilizing Multi-Modal, Multi-Lingual, and Multi-Feature Fusion}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Pham, Nhat Truong and Phan, Le Thi and Dang, Duc Ngoc Minh and Manavalan, Balachandran}</span><span class="p">,</span>
  <span class="na">booktitle</span> <span class="p">=</span> <span class="s">{Proceedings of the 12th International Symposium on Information and Communication Technology}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{870--877}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1145/3628797.3628887}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#215d42"> <a href="https://iniscom.eai-conferences.org/" rel="external nofollow noopener" target="_blank">INISCOM</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/INISCOM23.png" sizes="200px"></source> <img src="/assets/img/publication_preview/INISCOM23.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="INISCOM23.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="ranjbar2022rate" class="col-sm-8"> <div class="title">Multi-modal Speech Emotion Recognition: Improving Accuracy Through Fusion of VGGish and BERT Features with Multi-head Attention</div> <div class="author"> <a href="https://scholar.google.com/citations?hl=en&amp;user=NKbwDD8AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Phuong-Nam Tran</a> , Thuy-Duong Thi Vu , <a href="https://scholar.google.com/citations?hl=en&amp;user=2UKP440AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Duc Ngoc Minh Dang</a> , <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , and Anh-Khoa Tran </div> <div class="periodical"> <em>In <i><a href="https://iniscom.eai-conferences.org/" style="color: #00ab37; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">International Conference on Industrial Networks and Intelligent Systems</a></i></em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://link.springer.com/chapter/10.1007/978-3-031-47359-3_11" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1007/978-3-031-47359-3_11" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:UuEBAcK4md4C" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-5-4285F4?logo=googlescholar&amp;labelColor=beige" alt="5 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Recent research has shown that multi-modal learning is a successful method for enhancing classification performance by mixing several forms of input, notably in speech-emotion recognition (SER) tasks. However, the difference between the modalities may affect SER performance. To overcome this problem, a novel approach for multi-modal SER called 3M-SER is proposed in this paper. The 3M-SER leverages multi-head attention to fuse information from multiple feature embeddings, including audio and text features. The 3M-SER approach is based on the SERVER approach but includes an additional fusion module that improves the integration of text and audio features, leading to improved classification performance. To further enhance the correlation between the modalities, a LayerNorm is applied to audio features prior to fusion. Our approach achieved an unweighted accuracy (UA) and weighted accuracy (WA) of 79.96% and 80.66%, respectively, on the IEMOCAP benchmark dataset. This indicates that the proposed approach is better than SERVER and recent methods with similar approaches. In addition, it highlights the effectiveness of incorporating an extra fusion module in multi-modal learning.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@inproceedings</span><span class="p">{</span><span class="nl">ranjbar2022rate</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Multi-modal Speech Emotion Recognition: Improving Accuracy Through Fusion of VGGish and BERT Features with Multi-head Attention}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Tran, Phuong-Nam and Vu, Thuy-Duong Thi and Dang, Duc Ngoc Minh and Pham, Nhat Truong and Tran, Anh-Khoa}</span><span class="p">,</span>
  <span class="na">booktitle</span> <span class="p">=</span> <span class="s">{International Conference on Industrial Networks and Intelligent Systems}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{148--158}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">organization</span> <span class="p">=</span> <span class="s">{Springer}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/978-3-031-47359-3_11}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://www.sciencedirect.com/journal/computers-in-biology-and-medicine" rel="external nofollow noopener" target="_blank">CIBM</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/ADP-Fuse.png" sizes="200px"></source> <img src="/assets/img/publication_preview/ADP-Fuse.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="ADP-Fuse.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="basith2023adp" class="col-sm-8"> <div class="title"> <b><i>ADP-Fuse:</i></b> A novel dual layer machine learning predictor to identify antidiabetic peptides and diabetes types using multiview information</div> <div class="author"> <a href="https://scholar.google.com/citations?hl=en&amp;user=2eqv0J4AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Shaherin Basith</a> , <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , <a href="https://scholar.google.com/citations?hl=en&amp;user=sBSJfo4AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Minkyung Song</a> , Gwang Lee , and <a href="https://scholar.google.com/citations?hl=en&amp;user=0vkenbwAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Balachandran Manavalan</a> </div> <div class="periodical"> <em><i><a href="https://www.sciencedirect.com/journal/computers-in-biology-and-medicine" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Computers in Biology and Medicine</a></i></em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/abs/pii/S001048252300851X" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://balalab-skku.org/ADP-Fuse/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Website</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.compbiomed.2023.107386" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:ziOE8S1-AIUC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-17-4285F4?logo=googlescholar&amp;labelColor=beige" alt="17 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Diabetes mellitus has become a major public health concern associated with high mortality and reduced life expectancy and can cause blindness, heart attacks, kidney failure, lower limb amputations, and strokes. A new generation of antidiabetic peptides (ADPs) that act on β-cells or T-cells to regulate insulin production is being developed to alleviate the effects of diabetes. However, the lack of effective peptide-mining tools has hampered the discovery of these promising drugs. Hence, novel computational tools need to be developed urgently. In this study, we present ADP-Fuse, a novel two-layer prediction framework capable of accurately identifying ADPs or non-ADPs and categorizing them into type 1 and type 2 ADPs. First, we comprehensively evaluated 22 peptide sequence-derived features coupled with eight notable machine learning algorithms. Subsequently, the most suitable feature descriptors and classifiers for both layers were identified. The output of these single-feature models, embedded with multiview information, was trained with an appropriate classifier to provide the final prediction. Comprehensive cross-validation and independent tests substantiate that ADP-Fuse surpasses single-feature models and the feature fusion approach for the prediction of ADPs and their types. In addition, the SHapley Additive exPlanation method was used to elucidate the contributions of individual features to the prediction of ADPs and their types. Finally, a user-friendly web server for ADP-Fuse was developed and made publicly accessible (<a href="https://balalab-skku.org/ADP-Fuse/" rel="external nofollow noopener" target="_blank">https://balalab-skku.org/ADP-Fuse/</a>), enabling the swift screening and identification of novel ADPs and their types. This framework is expected to contribute significantly to antidiabetic peptide identification.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">basith2023adp</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{ADP-Fuse: A novel dual layer machine learning predictor to identify antidiabetic peptides and diabetes types using multiview information}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Basith, Shaherin and Pham, Nhat Truong and Song, Minkyung and Lee, Gwang and Manavalan, Balachandran}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Computers in Biology and Medicine}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{165}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{107386}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.compbiomed.2023.107386}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#215d42"> <a href="https://www.iciit.org/index.html" rel="external nofollow noopener" target="_blank">ICIIT</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/ICIIT23.png" sizes="200px"></source> <img src="/assets/img/publication_preview/ICIIT23.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="ICIIT23.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="pham2023server" class="col-sm-8"> <div class="title"> <b><i>SERVER:</i></b> Multi-modal Speech Emotion Recognition using TransformeR-based and Vision-based Embeddings</div> <div class="author"> <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , <a href="https://scholar.google.com/citations?hl=en&amp;user=2UKP440AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Duc Ngoc Minh Dang</a> , Bich Hong Ngoc Pham , and <a href="https://scholar.google.com/citations?hl=en&amp;user=XFUxOkoAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Sy Dzung Nguyen</a> </div> <div class="periodical"> <em>In <i><a href="https://www.iciit.org/index.html" style="color: #00ab37; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Proceedings of the 2023 8th International Conference on Intelligent Information Technology</a></i></em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://dl.acm.org/doi/10.1145/3591569.3591610" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://github.com/nhattruongpham/mmser" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1145/3591569.3591610" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:-nhnvRiOwuoC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-10-4285F4?logo=googlescholar&amp;labelColor=beige" alt="10 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>This paper proposes a multi-modal approach for speech emotion recognition (SER) using both text and audio inputs. The audio embedding is extracted by using a vision-based architecture, namely VGGish, while the text embedding is extracted by using a transformer-based architecture, namely BERT. Then, these embeddings are fused using concatenation to recognize emotional states. To evaluate the effectiveness of the proposed method, the benchmark dataset, namely IEMOCAP, is employed in this study. Experimental results indicate that the proposed method is very competitive and better than most of the latest and state-of-the-art methods using multi-modal analysis for SER. The proposed method achieves 63.00% unweighted accuracy (UA) and 63.10% weighted accuracy (WA) on the IEMOCAP dataset. In the future, an extension of multi-task learning and multi-lingual approaches will be investigated to improve the performance and robustness of multi-modal SER. For reproducibility purposes, our code is publicly available.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@inproceedings</span><span class="p">{</span><span class="nl">pham2023server</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{SERVER: Multi-modal Speech Emotion Recognition using TransformeR-based and Vision-based Embeddings}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Pham, Nhat Truong and Dang, Duc Ngoc Minh and Pham, Bich Hong Ngoc and Nguyen, Sy Dzung}</span><span class="p">,</span>
  <span class="na">booktitle</span> <span class="p">=</span> <span class="s">{Proceedings of the 2023 8th International Conference on Intelligent Information Technology}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{234--238}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1145/3591569.3591610}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#215d42"> <a href="https://www.iciit.org/index.html" rel="external nofollow noopener" target="_blank">ICIIT</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/UR-MAC.png" sizes="200px"></source> <img src="/assets/img/publication_preview/UR-MAC.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="UR-MAC.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="dang2023uplink" class="col-sm-8"> <div class="title">Uplink registration-based MAC protocol for IEEE 802.11ah networks</div> <div class="author"> <a href="https://scholar.google.com/citations?hl=en&amp;user=2UKP440AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Duc Ngoc Minh Dang</a> , and <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> </div> <div class="periodical"> <em>In <i><a href="https://www.iciit.org/index.html" style="color: #00ab37; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Proceedings of the 2023 8th International Conference on Intelligent Information Technology</a></i></em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://dl.acm.org/doi/10.1145/3591569.3591575" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1145/3591569.3591575" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:F2UWTTQJPOcC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-0-4285F4?logo=googlescholar&amp;labelColor=beige" alt="0 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>IEEE 802.11ah (Wi-Fi HaLow) operates in license-exempt ISM bands below 1 GHz and provides longer-range connectivity. The main advantage of the IEEE 802.11ah is it provides long range connection with low power consumption. RAW (Restricted Access Window) in IEEE 802.11ah helps to reduce the collision probability and enhance the network throughput when many stations contend the channel. Since stations are assigned to uplink RAW slots based on their Association Identifications (AID), the number of stations that have uplink data packets in each RAW slot is a big difference. It results in low fairness among stations. The paper proposes an uplink registration-based MAC protocol for IEEE 802.11ah networks (UR-MAC). In UR-MAC protocol, stations with uplink data will register with the AP by attaching the uplink registration to the data packet during downlink communications. The AP will allocate RAW slots based on the uplink registered station list. The UR-MAC protocol tries to use up the resources of the RAW slots as well as balance the number of stations with uplink data among the RAW slots. Through the evaluation and comparison analysis, the UR-MAC protocol significantly improves the fairness index compared to the IEEE 802.11ah protocol while still ensuring the probability of successful transmission, the average number of successfully transmitted packets, and power efficiency compared to the IEEE 802.11ah protocol.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@inproceedings</span><span class="p">{</span><span class="nl">dang2023uplink</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Uplink registration-based MAC protocol for IEEE 802.11ah networks}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Dang, Duc Ngoc Minh and Pham, Nhat Truong}</span><span class="p">,</span>
  <span class="na">booktitle</span> <span class="p">=</span> <span class="s">{Proceedings of the 2023 8th International Conference on Intelligent Information Technology}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{33--37}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1145/3591569.3591575}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://www.sciencedirect.com/journal/computers-in-biology-and-medicine" rel="external nofollow noopener" target="_blank">CIBM</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/DrugormerDTI.png" sizes="200px"></source> <img src="/assets/img/publication_preview/DrugormerDTI.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="DrugormerDTI.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="hu2023drugormerdti" class="col-sm-8"> <div class="title"> <b><i>DrugormerDTI:</i></b> Drug Graphormer for drug–target interaction prediction</div> <div class="author"> Jiayue Hu , Wang Yu , Chao Pang , <a href="https://scholar.google.com/citations?hl=en&amp;user=7Lw6yNYAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Junru Jin</a> , <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , <a href="https://scholar.google.com/citations?hl=en&amp;user=0vkenbwAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Balachandran Manavalan</a> , and <a href="https://scholar.google.com/citations?hl=en&amp;user=0EAV03MAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Leyi Wei</a> </div> <div class="periodical"> <em><i><a href="https://www.sciencedirect.com/journal/computers-in-biology-and-medicine" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Computers in Biology and Medicine</a></i></em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/pii/S0010482523004110" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://github.com/joannacatj/drugormerDTI" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.compbiomed.2023.106946" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:jE2MZjpN3IcC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-10-4285F4?logo=googlescholar&amp;labelColor=beige" alt="10 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Drug-target interactions (DTI) prediction is a crucial task in drug discovery. Existing computational methods accelerate the drug discovery in this respect. However, most of them suffer from low feature representation ability, significantly affecting the predictive performance. To address the problem, we propose a novel neural network architecture named DrugormerDTI, which uses Graph Transformer to learn both sequential and topological information through the input molecule graph and Resudual2vec to learn the underlying relation between residues from proteins. By conducting ablation experiments, we verify the importance of each part of the DrugormerDTI. We also demonstrate the good feature extraction and expression capabilities of our model via comparing the mapping results of the attention layer and molecular docking results. Experimental results show that our proposed model performs better than baseline methods on four benchmarks. We demonstrate that the introduction of Graph Transformer and the design of residue are appropriate for drug-target prediction.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">hu2023drugormerdti</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{DrugormerDTI: Drug Graphormer for drug--target interaction prediction}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Hu, Jiayue and Yu, Wang and Pang, Chao and Jin, Junru and Pham, Nhat Truong and Manavalan, Balachandran and Wei, Leyi}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Computers in Biology and Medicine}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{161}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{106946}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.compbiomed.2023.106946}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://www.sciencedirect.com/journal/knowledge-based-systems" rel="external nofollow noopener" target="_blank">KBS</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/AAD-Net.png" sizes="200px"></source> <img src="/assets/img/publication_preview/AAD-Net.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="AAD-Net.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="mustaqeem2023aad" class="col-sm-8"> <div class="title"> <b><i>AAD-Net:</i></b> Advanced end-to-end signal processing system for human emotion detection &amp; recognition using attention-based deep echo state network</div> <div class="author"> <a href="https://scholar.google.com/citations?hl=en&amp;user=uEZhRWAAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Mustaqeem Khan</a> , <a href="https://scholar.google.com/citations?hl=en&amp;user=VcOjgngAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Abdulmotaleb El Saddik</a> , Fahd Saleh Alotaibi , and <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> </div> <div class="periodical"> <em><i><a href="https://www.sciencedirect.com/journal/knowledge-based-systems" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Knowledge-Based Systems</a></i></em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/abs/pii/S0950705123002757" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.knosys.2023.110525" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:HGTzPopzzJcC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-49-4285F4?logo=googlescholar&amp;labelColor=beige" alt="49 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Speech signals are the most convenient way of communication between human beings and the eventual method of Human-Computer Interaction (HCI) to exchange emotions and information. Recognizing emotions from speech signals is a challenging task due to the sparse nature of emotional data and features. In this article, we proposed a Deep Echo-State-Network (DeepESN) system for emotion recognition with a dilated convolution neural network and multi-headed attention mechanism. To reduce the model complexity, we incorporate a DeepESN that combines reservoir computing for higher-dimensional mapping. We also used fine-tuned Sparse Random Projection (SRP) to reduce dimensionality and adopted an early fusion strategy to fuse the extracted cues and passed the joint feature vector via a classification layer to recognize emotions. Our proposed model is evaluated on two public speech corpora, EMO-DB and RAVDESS, and tested for subject/speaker-dependent/independent performance. The results show that our proposed system achieves a high recognition rate, 91.14, 85.57 for EMO-DB, and 82.01, 77.02 for RAVDESS, using speaker-dependent and independent experiments, respectively. Our proposed system outperforms the State-of-The-Art (SOTA) while requiring less computational time.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">mustaqeem2023aad</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{AAD-Net: Advanced end-to-end signal processing system for human emotion detection \&amp; recognition using attention-based deep echo state network}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Khan, Mustaqeem and El Saddik, Abdulmotaleb and Alotaibi, Fahd Saleh and Pham, Nhat Truong}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Knowledge-Based Systems}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{270}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{110525}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.knosys.2023.110525}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://link.springer.com/journal/500" rel="external nofollow noopener" target="_blank">SOCO</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/StockML.png" sizes="200px"></source> <img src="/assets/img/publication_preview/StockML.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="StockML.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="kumar2023towards" class="col-sm-8"> <div class="title">Towards an efficient machine learning model for financial time series forecasting</div> <div class="author"> Arun Kumar , Tanya Chauhan , <a href="https://scholar.google.com/citations?hl=en&amp;user=WPdTWisAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Srinivasan Natesan</a> , <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , <a href="https://scholar.google.com/citations?hl=en&amp;user=At-Y-H8AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Ngoc Duy Nguyen</a> , and <a href="https://scholar.google.com/citations?hl=en&amp;user=an_4IJkAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Chee Peng Lim</a> </div> <div class="periodical"> <em><i><a href="https://link.springer.com/journal/500" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Soft Computing</a></i></em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://link.springer.com/article/10.1007/s00500-023-08676-x" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1007/s00500-023-08676-x" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:ce2CqMG-AY4C" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-4-4285F4?logo=googlescholar&amp;labelColor=beige" alt="4 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Financial time series forecasting is a challenging problem owing to the high degree of randomness and absence of residuals in time series data. Existing machine learning solutions normally do not perform well on such data. In this study, we propose an efficient machine learning model for financial time series forecasting through carefully designed feature extraction, elimination, and selection strategies. We leverage a binary particle swarm optimization algorithm to select the appropriate features and propose new evaluation metrics, i.e. mean weighted square error and mean weighted square ratio, for better performance assessment in handling financial time series data. Both indicators ascertain that our proposed model is effective, which outperforms several existing methods in benchmark studies.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">kumar2023towards</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Towards an efficient machine learning model for financial time series forecasting}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Kumar, Arun and Chauhan, Tanya and Natesan, Srinivasan and Pham, Nhat Truong and Nguyen, Ngoc Duy and Lim, Chee Peng}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Soft Computing}</span><span class="p">,</span>
  <span class="na">volumev</span> <span class="p">=</span> <span class="s">{27}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{11329--11339}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Springer}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/s00500-023-08676-x}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://www.sciencedirect.com/journal/expert-systems-with-applications" rel="external nofollow noopener" target="_blank">ESWA</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/HDA_mADCRNN.png" sizes="200px"></source> <img src="/assets/img/publication_preview/HDA_mADCRNN.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="HDA_mADCRNN.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="pham2023hybrid" class="col-sm-8"> <div class="title">Hybrid data augmentation and deep attention-based dilated convolutional-recurrent neural networks for speech emotion recognition</div> <div class="author"> <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , <a href="https://scholar.google.com/citations?hl=en&amp;user=2UKP440AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Duc Ngoc Minh Dang</a> , <a href="https://scholar.google.com/citations?hl=en&amp;user=At-Y-H8AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Ngoc Duy Nguyen</a> , <a href="https://scholar.google.com/citations?hl=en&amp;user=xr39SOwAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Thanh Thi Nguyen</a> , <a href="https://scholar.google.com/citations?hl=en&amp;user=5b9ncWoAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Hai Nguyen</a> , <a href="https://scholar.google.com/citations?hl=en&amp;user=0vkenbwAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Balachandran Manavalan</a> , <a href="https://scholar.google.com/citations?hl=en&amp;user=an_4IJkAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Chee Peng Lim</a> , and <a href="https://scholar.google.com/citations?hl=en&amp;user=XFUxOkoAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Sy Dzung Nguyen</a> </div> <div class="periodical"> <em><i><a href="https://www.sciencedirect.com/journal/expert-systems-with-applications" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Expert Systems with Applications</a></i></em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://arxiv.org/abs/2109.09026" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">arXiv</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/pii/S0957417423011107" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/HDA_mADCRNN.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a> <a href="https://github.com/nhattruongpham/hda-adcrnn-ser" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.eswa.2023.120608" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:NDuN12AVoxsC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-39-4285F4?logo=googlescholar&amp;labelColor=beige" alt="39 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Recently, speech emotion recognition (SER) has become an active research area in speech processing, particularly with the advent of deep learning (DL). Numerous DL-based methods have been proposed for SER. However, most of the existing DL-based models are complex and require a large amounts of data to achieve a good performance. In this study, a new framework of deep attention-based dilated convolutional-recurrent neural networks coupled with a hybrid data augmentation method was proposed for addressing SER tasks. The hybrid data augmentation method constitutes an upsampling technique for generating more speech data samples based on the traditional and generative adversarial network approaches. By leveraging both convolutional and recurrent neural networks in a dilated form along with an attention mechanism, the proposed DL framework can extract high-level representations from three-dimensional log Mel spectrogram features. Dilated convolutional neural networks acquire larger receptive fields, whereas dilated recurrent neural networks overcome complex dependencies as well as the vanishing and exploding gradient issues. Furthermore, the loss functions are reconfigured by combining the SoftMax loss and the center-based losses to classify various emotional states. The proposed framework was implemented using the Python programming language and the TensorFlow deep learning library. To validate the proposed framework, the EmoDB and ERC benchmark datasets, which are imbalanced and/or small datasets, were employed. The experimental results indicate that the proposed framework outperforms other related state-of-the-art methods, yielding the highest unweighted recall rates of 88.03 ± 1.39 (%) and 66.56 ± 0.67 (%) for the EmoDB and ERC datasets, respectively.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">pham2023hybrid</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Hybrid data augmentation and deep attention-based dilated convolutional-recurrent neural networks for speech emotion recognition}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Pham, Nhat Truong and Dang, Duc Ngoc Minh and Nguyen, Ngoc Duy and Nguyen, Thanh Thi and Nguyen, Hai and Manavalan, Balachandran and Lim, Chee Peng and Nguyen, Sy Dzung}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Expert Systems with Applications}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{120608}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.eswa.2023.120608}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://www.sciencedirect.com/journal/international-journal-of-biological-macromolecules" rel="external nofollow noopener" target="_blank">IJBIOMAC</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/Pretoria.png" sizes="200px"></source> <img src="/assets/img/publication_preview/Pretoria.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="Pretoria.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="charoenkwan2023pretoria" class="col-sm-8"> <div class="title"> <b><i>Pretoria:</i></b> An effective computational approach for accurate and high-throughput identification of CD8<sup>+</sup> t-cell epitopes of eukaryotic pathogens</div> <div class="author"> Phasit Charoenkwan , Nalini Schaduangrat , <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , <a href="https://scholar.google.com/citations?hl=en&amp;user=0vkenbwAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Balachandran Manavalan</a> , and Watshara Shoombuatong </div> <div class="periodical"> <em><i><a href="https://www.sciencedirect.com/journal/international-journal-of-biological-macromolecules" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">International Journal of Biological Macromolecules</a></i></em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/abs/pii/S0141813023011224" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="http://pmlabstack.pythonanywhere.com/Pretoria" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Website</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.ijbiomac.2023.124228" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:6_hjMsCP8ZoC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-4-4285F4?logo=googlescholar&amp;labelColor=beige" alt="4 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>T-cells recognize antigenic epitopes present on major histocompatibility complex (MHC) molecules, triggering an adaptive immune response in the host. T-cell epitope (TCE) identification is challenging because of the extensive number of undetermined proteins found in eukaryotic pathogens, as well as MHC polymorphisms. In addition, conventional experimental approaches for TCE identification are time-consuming and expensive. Thus, computational approaches that can accurately and rapidly identify CD8<sup>+</sup> T-cell epitopes (TCEs) of eukaryotic pathogens based solely on sequence information may facilitate the discovery of novel CD8<sup>+</sup> TCEs in a cost-effective manner. Here, Pretoria (Predictor of CD8<sup>+</sup> TCEs of eukaryotic pathogens) is proposed as the first stack-based approach for accurate and large-scale identification of CD8<sup>+</sup> TCEs of eukaryotic pathogens. In particular, Pretoria enabled the extraction and exploration of crucial information embedded in CD8<sup>+</sup> TCEs by employing a comprehensive set of 12 well-known feature descriptors extracted from multiple groups, including physicochemical properties, composition-transition-distribution, pseudo-amino acid composition, and amino acid composition. These feature descriptors were then utilized to construct a pool of 144 different machine learning (ML)-based classifiers based on 12 popular ML algorithms. Finally, the feature selection method was used to effectively determine the important ML classifiers for the construction of our stacked model. The experimental results indicated that Pretoria is an accurate and effective computational approach for CD8<sup>+</sup> TCE prediction; it was superior to several conventional ML classifiers and the existing method in terms of the independent test, with an accuracy of 0.866, MCC of 0.732, and AUC of 0.921. Additionally, to maximize user convenience for high-throughput identification of CD8<sup>+</sup> TCEs of eukaryotic pathogens, a user-friendly web server of Pretoria (<a href="http://pmlabstack.pythonanywhere.com/Pretoria" rel="external nofollow noopener" target="_blank">http://pmlabstack.pythonanywhere.com/Pretoria</a>) was developed and made freely available.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">charoenkwan2023pretoria</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Pretoria: An effective computational approach for accurate and high-throughput identification of CD8+ t-cell epitopes of eukaryotic pathogens}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Charoenkwan, Phasit and Schaduangrat, Nalini and Pham, Nhat Truong and Manavalan, Balachandran and Shoombuatong, Watshara}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{International Journal of Biological Macromolecules}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{238}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{124228}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.ijbiomac.2023.124228}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://www.cell.com/heliyon/home" rel="external nofollow noopener" target="_blank">Heliyon</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/Humain_brain.png" sizes="200px"></source> <img src="/assets/img/publication_preview/Humain_brain.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="Humain_brain.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="pham2023exploratory" class="col-sm-8"> <div class="title">An exploratory simulation study and prediction model on human brain behavior and activity using an integration of deep neural network and biosensor Rabi antenna</div> <div class="author"> <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , Montree Bunruangses , Phichai Youplao , Anita Garhwal , Kanad Ray , Arup Roy , Sarawoot Boonkirdram , Preecha Yupapin , Muhammad Arif Jalil , Jalil Ali , <a href="https://scholar.google.com/citations?hl=en&amp;user=yjrSXiEAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Shamim Kaiser</a> , <a href="https://scholar.google.com/citations?hl=en&amp;user=L8em2YoAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Mufti Mahmud</a> , <a href="https://scholar.google.com/citations?hl=en&amp;user=e-6kvkIAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Saurav Mallik</a> , and <a href="https://scholar.google.com/citations?hl=en&amp;user=eDRMnHQAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Zhongming Zhao</a> </div> <div class="periodical"> <em><i><a href="https://www.cell.com/heliyon/home" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Heliyon</a></i></em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.cell.com/heliyon/fulltext/S2405-8440(23)02956-0" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/Deep_Brain_SigNet.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a> <a href="https://github.com/nhattruongpham/Deep_Brain_SigNet" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.heliyon.2023.e15749" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:TlpoogIpr_IC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-4-4285F4?logo=googlescholar&amp;labelColor=beige" alt="4 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>The plasmonic antenna probe is constructed using a silver rod embedded in a modified Mach-Zehnder interferometer (MZI) ad-drop filter. Rabi antennas are formed when space-time control reaches two levels of system oscillation and can be used as human brain sensor probes. Photonic neural networks are designed using brain-Rabi antenna communication, and transmissions are connected via neurons. Communication signals are carried by electron spin (up and down) and adjustable Rabi frequency. Hidden variables and deep brain signals can be obtained by external detection. A Rabi antenna has been developed by simulation using computer simulation technology (CST) software. Additionally, a communication device has been developed that uses the Optiwave program with Finite-Difference Time-Domain (OptiFDTD). The output signal is plotted using the MATLAB program with the parameters of the OptiFDTD simulation results. The proposed antenna oscillates in the frequency range of 192 THz to 202 THz with a maximum gain of 22.4 dBi. The sensitivity of the sensor is calculated along with the result of electron spin and applied to form a human brain connection. Moreover, intelligent machine learning algorithms are proposed to identify high-quality transmissions and predict the behavior of transmissions in the near future. During the process, a root mean square error (RMSE) of 2.3332 (±0.2338) was obtained. Finally, it can be said that our proposed model can efficiently predict human mind, thoughts, behavior as well as action/reaction, which can be greatly helpful in the diagnosis of various neuro-degenerative/psychological diseases (such as Alzheimer’s, dementia, etc.) and for security purposes.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">pham2023exploratory</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{An exploratory simulation study and prediction model on human brain behavior and activity using an integration of deep neural network and biosensor Rabi antenna}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Pham, Nhat Truong and Bunruangses, Montree and Youplao, Phichai and Garhwal, Anita and Ray, Kanad and Roy, Arup and Boonkirdram, Sarawoot and Yupapin, Preecha and Jalil, Muhammad Arif and Ali, Jalil and Kaiser, Shamim and Mahmud, Mufti and Mallik, Saurav and Zhao, Zhongming}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Heliyon}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{9}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{5}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.heliyon.2023.e15749}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://www.tandfonline.com/journals/tjit20" rel="external nofollow noopener" target="_blank">T&amp;F JIT</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/osw-1d-prn-shap.png" sizes="200px"></source> <img src="/assets/img/publication_preview/osw-1d-prn-shap.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="osw-1d-prn-shap.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="pham2023speech" class="col-sm-8"> <div class="title">Speech emotion recognition using overlapping sliding window and Shapley additive explainable deep neural network</div> <div class="author"> <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , <a href="https://scholar.google.com/citations?hl=en&amp;user=XFUxOkoAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Sy Dzung Nguyen</a> , Vu Song Thuy Nguyen , Bich Ngoc Hong Pham , and <a href="https://scholar.google.com/citations?hl=en&amp;user=2UKP440AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Duc Ngoc Minh Dang</a> </div> <div class="periodical"> <em><i><a href="https://www.tandfonline.com/journals/tjit20" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Journal of Information and Telecommunication</a></i></em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.tandfonline.com/doi/full/10.1080/24751839.2023.2187278" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/osw-1d-prn-shap.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a> <a href="https://github.com/nhattruongpham/osw-1d-prn-shap" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1080/24751839.2023.2187278" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:q-HalDI95KYC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-11-4285F4?logo=googlescholar&amp;labelColor=beige" alt="11 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Speech emotion recognition (SER) has several applications, such as e-learning, human-computer interaction, customer service, and healthcare systems. Although researchers have investigated lots of techniques to improve the accuracy of SER, it has been challenging with feature extraction, classifier schemes, and computational costs. To address the aforementioned problems, we propose a new set of 1D features extracted by using an overlapping sliding window (OSW) technique for SER in this study. In addition, a deep neural network-based classifier scheme called the deep Pattern Recognition Network (PRN) is designed to categorize emotional states from the new set of 1D features. We evaluate the proposed method on the Emo-DB and the AESSD datasets that contain several different emotional states. The experimental results show that the proposed method achieves an accuracy of 98.5% and 87.1% on the Emo-DB and AESSD datasets, respectively. It is also more comparable with accuracy to and better than the state-of-the-art and current approaches that use 1D features on the same datasets for SER. Furthermore, the SHAP (SHapley Additive exPlanations) analysis is employed for interpreting the prediction model to assist system developers in selecting the optimal features to integrate into the desired system.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">pham2023speech</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Speech emotion recognition using overlapping sliding window and Shapley additive explainable deep neural network}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Pham, Nhat Truong and Nguyen, Sy Dzung and Nguyen, Vu Song Thuy and Pham, Bich Ngoc Hong and Dang, Duc Ngoc Minh}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Journal of Information and Telecommunication}</span><span class="p">,</span>
  <span class="na">volumevo</span> <span class="p">=</span> <span class="s">{7}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{317--335}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Taylor &amp; Francis}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1080/24751839.2023.2187278}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://www.sciencedirect.com/journal/expert-systems-with-applications" rel="external nofollow noopener" target="_blank">ESWA</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/Fruit-CoV.png" sizes="200px"></source> <img src="/assets/img/publication_preview/Fruit-CoV.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="Fruit-CoV.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="nguyen2023fruit" class="col-sm-8"> <div class="title"> <b><i>Fruit-CoV:</i></b> An efficient vision-based framework for speedy detection and diagnosis of SARS-CoV-2 infections through recorded cough sounds</div> <div class="author"> Long H. Nguyen<sup>†</sup> , <em><b style="color: #FFA500;">Nhat Truong Pham<sup>†*</sup></b></em> , Van Huong Do , Liu Tai Nguyen , <a href="https://scholar.google.com/citations?hl=en&amp;user=zSAfD80AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Thanh Tin Nguyen</a> , <a href="https://scholar.google.com/citations?hl=en&amp;user=5b9ncWoAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Hai Nguyen</a> , <a href="https://scholar.google.com/citations?hl=en&amp;user=At-Y-H8AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Ngoc Duy Nguyen</a> , <a href="https://scholar.google.com/citations?hl=en&amp;user=xr39SOwAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Thanh Thi Nguyen</a> , <a href="https://scholar.google.com/citations?hl=en&amp;user=XFUxOkoAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Sy Dzung Nguyen</a> , Asim Bhatti , and <a href="https://scholar.google.com/citations?hl=en&amp;user=an_4IJkAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Chee Peng Lim</a> </div> <div class="periodical"> <em><i><a href="https://www.sciencedirect.com/journal/expert-systems-with-applications" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Expert Systems with Applications</a></i></em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://arxiv.org/abs/2109.03219" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">arXiv</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.sciencedirect.com/science/article/pii/S0957417422022308" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://github.com/nhattruongpham/Fruit-CoV" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1016/j.eswa.2022.119212" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:WHdLCjDvYFkC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-16-4285F4?logo=googlescholar&amp;labelColor=beige" alt="16 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>COVID-19 is an infectious disease caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). This deadly virus has spread worldwide, leading to a global pandemic since March 2020. A recent variant of SARS-CoV-2 named Delta is intractably contagious and responsible for more than four million deaths globally. Therefore, developing an efficient self-testing service for SARS-CoV-2 at home is vital. In this study, a two-stage vision-based framework, namely Fruit-CoV, is introduced for detecting SARS-CoV-2 infections through recorded cough sounds. Specifically, audio signals are converted into Log-Mel spectrograms, and the EfficientNet-V2 network is used to extract their visual features in the first stage. In the second stage, 14 convolutional layers extracted from the large-scale Pretrained Audio Neural Networks for audio pattern recognition (PANNs) and the Wavegram-Log-Mel-CNN are employed to aggregate feature representations of the Log-Mel spectrograms and the waveform. Finally, the combined features are used to train a binary classifier. In this study, a dataset provided by the AICovidVN 115M Challenge is employed for evaluation. It includes 7,371 recorded cough sounds collected throughout Vietnam, India, and Switzerland. Experimental results indicate that the proposed model achieves an Area Under the Receiver Operating Characteristic Curve (AUC) score of 92.8% and ranks first on the final leaderboard of the AICovidVN 115M Challenge. Our code is publicly available.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">nguyen2023fruit</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Fruit-CoV: An efficient vision-based framework for speedy detection and diagnosis of SARS-CoV-2 infections through recorded cough sounds}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Nguyen, Long H. and Pham, Nhat Truong and Do, Van Huong and Nguyen, Liu Tai and Nguyen, Thanh Tin and Nguyen, Hai and Nguyen, Ngoc Duy and Nguyen, Thanh Thi and Nguyen, Sy Dzung and Bhatti, Asim and Lim, Chee Peng}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Expert Systems with Applications}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{213}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{119212}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Elsevier}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1016/j.eswa.2022.119212}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://link.springer.com/journal/10489" rel="external nofollow noopener" target="_blank">APIN</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/Fruit-API.png" sizes="200px"></source> <img src="/assets/img/publication_preview/Fruit-API.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="Fruit-API.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="nguyen2023towards" class="col-sm-8"> <div class="title">Towards designing a generic and comprehensive deep reinforcement learning framework</div> <div class="author"> <a href="https://scholar.google.com/citations?hl=en&amp;user=At-Y-H8AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Ngoc Duy Nguyen</a> , <a href="https://scholar.google.com/citations?hl=en&amp;user=xr39SOwAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Thanh Thi Nguyen</a> , <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , <a href="https://scholar.google.com/citations?hl=en&amp;user=5b9ncWoAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Hai Nguyen</a> , Dang Tu Nguyen , Thanh Dang Nguyen , <a href="https://scholar.google.com/citations?hl=en&amp;user=an_4IJkAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Chee Peng Lim</a> , Michael Johnstone , Asim Bhatti , Douglas Creighton , and <a href="https://scholar.google.com/citations?hl=en&amp;user=_VIKLjEAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Saeid Nahavandi</a> </div> <div class="periodical"> <em><i><a href="https://link.springer.com/journal/10489" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Applied Intelligence</a></i></em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://link.springer.com/article/10.1007/s10489-022-03550-z" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/Fruit-API.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a> <a href="https://github.com/garlicdevs/Fruit-API" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> <a href="https://fruitlab.org/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Website</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1007/s10489-022-03550-z" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:mKu_rENv82IC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-5-4285F4?logo=googlescholar&amp;labelColor=beige" alt="5 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Reinforcement learning (RL) has emerged as an effective approach for building an intelligent system, which involves multiple self-operated agents to collectively accomplish a designated task. More importantly, there has been a renewed focus on RL since the introduction of deep learning that essentially makes RL feasible to operate in high-dimensional environments. However, there are many diversified research directions in the current literature, such as multi-agent and multi-objective learning, and human-machine interactions. Therefore, in this paper, we propose a comprehensive software architecture that not only plays a vital role in designing a connect-the-dots deep RL architecture but also provides a guideline to develop a realistic RL application in a short time span. By inheriting the proposed architecture, software managers can foresee any challenges when designing a deep RL-based system. As a result, they can expedite the design process and actively control every stage of software development, which is especially critical in agile development environments. For this reason, we design a deep RL-based framework that strictly ensures flexibility, robustness, and scalability. To enforce generalization, the proposed architecture also does not depend on a specific RL algorithm, a network configuration, the number of agents, or the type of agents.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">nguyen2023towards</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Towards designing a generic and comprehensive deep reinforcement learning framework}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Nguyen, Ngoc Duy and Nguyen, Thanh Thi and Pham, Nhat Truong and Nguyen, Hai and Nguyen, Dang Tu and Nguyen, Thanh Dang and Lim, Chee Peng and Johnstone, Michael and Bhatti, Asim and Creighton, Douglas and Nahavandi, Saeid}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Applied Intelligence}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{53}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{2967--2988}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2023}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Springer}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/s10489-022-03550-z}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#215d42"> <a href="https://link.springer.com/conference/aeta" rel="external nofollow noopener" target="_blank">AETA</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/3MTL_SER.png" sizes="200px"></source> <img src="/assets/img/publication_preview/3MTL_SER.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="3MTL_SER.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="pham2022speech" class="col-sm-8"> <div class="title">Speech emotion recognition: A brief review of multi-modal multi-task learning approaches</div> <div class="author"> <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , <a href="https://scholar.google.com/citations?hl=en&amp;user=xrvf7L4AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Anh-Tuan Tran</a> , Bich Ngoc Hong Pham , Hanh Dang-Ngoc , <a href="https://scholar.google.com/citations?hl=en&amp;user=XFUxOkoAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Sy Dzung Nguyen</a> , and <a href="https://scholar.google.com/citations?hl=en&amp;user=2UKP440AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Duc Ngoc Minh Dang</a> </div> <div class="periodical"> <em>In <i><a href="https://link.springer.com/conference/aeta" style="color: #00ab37; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">International Conference on Advanced Engineering Theory and Applications</a></i></em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://link.springer.com/chapter/10.1007/978-981-99-8703-0_50" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1007/978-981-99-8703-0_50" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:hSRAE-fF4OAC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-4-4285F4?logo=googlescholar&amp;labelColor=beige" alt="4 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Speech emotion recognition (SER) has become an attention-grabbing topic in recent years thanks to the development of deep learning in the field of speech processing. However, it is difficult to recognize an accurate emotional state from only speech signals. Therefore, researchers have investigated multi-modalities such as speech, visual, and text inputs to improve the emotional recognition rate of speech. In addition, to enhance the generalized deep learning models for SER, multi-task learning (MTL) strategies have also been applied in the past decade. In this paper, a brief and comprehensive review of multi-modal multi-task learning (3MTL) approaches for recognizing emotional states from speech signals is presented, including multi-modal SER, multi-task learning SER, and multi-modal multi-task learning SER. This paper also discusses about some problems that still need to be solved in 3MTL SER and gives some suggestions for the future.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@inproceedings</span><span class="p">{</span><span class="nl">pham2022speech</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Speech emotion recognition: A brief review of multi-modal multi-task learning approaches}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Pham, Nhat Truong and Tran, Anh-Tuan and Pham, Bich Ngoc Hong and Dang-Ngoc, Hanh and Nguyen, Sy Dzung and Dang, Duc Ngoc Minh}</span><span class="p">,</span>
  <span class="na">booktitle</span> <span class="p">=</span> <span class="s">{International Conference on Advanced Engineering Theory and Applications}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{605--615}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">organization</span> <span class="p">=</span> <span class="s">{Springer}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/978-981-99-8703-0_50}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#215d42"> <a href="https://link.springer.com/conference/aeta" rel="external nofollow noopener" target="_blank">AETA</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/PUT_MAC.png" sizes="200px"></source> <img src="/assets/img/publication_preview/PUT_MAC.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="PUT_MAC.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="dang2022priority" class="col-sm-8"> <div class="title">Priority-Based Uplink Raw Slot Utilization in the IEEE 802.11 ah Networks</div> <div class="author"> <a href="https://scholar.google.com/citations?hl=en&amp;user=2UKP440AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Duc Ngoc Minh Dang</a> , Anh Khoa Tran , and <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> </div> <div class="periodical"> <em>In <i><a href="https://link.springer.com/conference/aeta" style="color: #00ab37; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">International Conference on Advanced Engineering Theory and Applications</a></i></em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://link.springer.com/chapter/10.1007/978-981-99-8703-0_12" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1007/978-981-99-8703-0_12" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:qwy9JoKyICEC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-0-4285F4?logo=googlescholar&amp;labelColor=beige" alt="0 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>The IEEE 802.11ah standard allows an Access Point (AP) to connect up to 8192 stations at a transmission range of up to 1 km. The goal of the IEEE 802.11ah standard is to maintain wide connectivity and energy efficiency. Some stations are allocated to RAW slots, but they do not have uplink data packets to transmit results in low channel efficiency. The new MAC protocol (PUT-MAC) allows stations to use adjacent RAW slots in a priority manner to improve channel access usage efficiency. The stations use different Arbitration Inter Frame Space and contention window values to contend channel in the same RAW slot. The paper conducts simulations to compare the network performance of the PUT-MAC protocol with the IEEE 802.11ah.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@inproceedings</span><span class="p">{</span><span class="nl">dang2022priority</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Priority-Based Uplink Raw Slot Utilization in the IEEE 802.11 ah Networks}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Dang, Duc Ngoc Minh and Tran, Anh Khoa and Pham, Nhat Truong}</span><span class="p">,</span>
  <span class="na">booktitle</span> <span class="p">=</span> <span class="s">{International Conference on Advanced Engineering Theory and Applications}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{143--151}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">organization</span> <span class="p">=</span> <span class="s">{Springer}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1007/978-981-99-8703-0_12}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://jcsce.vnu.edu.vn/index.php/jcsce" rel="external nofollow noopener" target="_blank">JCSCE</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/VNU_JCSCE.png" sizes="200px"></source> <img src="/assets/img/publication_preview/VNU_JCSCE.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="VNU_JCSCE.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="tin2022viecap4h" class="col-sm-8"> <div class="title"> <b><i>vieCap4H Challenge 2021:</i></b> Vietnamese Image Captioning for Healthcare Domain using Swin Transformer and Attention-based LSTM</div> <div class="author"> <a href="https://scholar.google.com/citations?hl=en&amp;user=zSAfD80AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Thanh Tin Nguyen</a> , Long H. Nguyen , <em><b style="color: #FFA500;">Nhat Truong Pham<sup>*</sup></b></em> , Liu Tai Nguyen , Van Huong Do , <a href="https://scholar.google.com/citations?hl=en&amp;user=5b9ncWoAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Hai Nguyen</a> , and <a href="https://scholar.google.com/citations?hl=en&amp;user=At-Y-H8AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Ngoc Duy Nguyen</a> </div> <div class="periodical"> <em><i><a href="https://jcsce.vnu.edu.vn/index.php/jcsce" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">VNU Journal of Science: Computer Science and Communication Engineering</a></i></em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="http://arxiv.org/abs/2209.01304" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">arXiv</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://jcsce.vnu.edu.vn/index.php/jcsce/article/view/369" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://github.com/ngthanhtin/VLSP_ImageCaptioning" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.25073/2588-1086/vnucsce.369" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:v6i8RKmR8ToC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-1-4285F4?logo=googlescholar&amp;labelColor=beige" alt="1 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>This study presents our approach on the automatic Vietnamese image captioning for healthcare domain in text processing tasks of Vietnamese Language and Speech Processing (VLSP) Challenge 2021, as shown in Figure 1. In recent years, image captioning often employs a convolutional neural network-based architecture as an encoder and a long short-term memory (LSTM) as a decoder to generate sentences. These models perform remarkably well in different datasets. Our proposed model also has an encoder and a decoder, but we instead use a Swin Transformer in the encoder, and a LSTM combined with an attention module in the decoder. The study presents our training experiments and techniques used during the competition. Our model achieves a BLEU4 score of 0.293 on the vietCap4H dataset, and the score is ranked the 3<sup>rd</sup> place on the private leaderboard. Our code can be found at <a href="https://github.com/ngthanhtin/VLSP_ImageCaptioning/" rel="external nofollow noopener" target="_blank">https://github.com/ngthanhtin/VLSP_ImageCaptioning/</a> for reproducible purposes.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">tin2022viecap4h</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{vieCap4H Challenge 2021: Vietnamese Image Captioning for Healthcare Domain using Swin Transformer and Attention-based LSTM}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Nguyen, Thanh Tin and Nguyen, Long H. and Pham, Nhat Truong and Nguyen, Liu Tai and Do, Van Huong and Nguyen, Hai and Nguyen, Ngoc Duy}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{VNU Journal of Science: Computer Science and Communication Engineering}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{38}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{2}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.25073/2588-1086/vnucsce.369}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#215d42"> <a href="https://atc-conf.org/" rel="external nofollow noopener" target="_blank">ATC</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/SF-MAC.png" sizes="200px"></source> <img src="/assets/img/publication_preview/SF-MAC.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="SF-MAC.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="dang2022space" class="col-sm-8"> <div class="title">Space-Frequency Diversity based MAC protocol for IEEE 802.11 ah networks</div> <div class="author"> <a href="https://scholar.google.com/citations?hl=en&amp;user=2UKP440AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Duc Ngoc Minh Dang</a> , Van Thau Tran , Hoang Lam Nguyen , <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , Anh Khoa Tran , and Ngoc-Hanh Dang </div> <div class="periodical"> <em>In <i><a href="https://atc-conf.org/" style="color: #00ab37; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">2022 International Conference on Advanced Technologies for Communications (ATC)</a></i></em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://ieeexplore.ieee.org/abstract/document/9943042" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1109/ATC55345.2022.9943042" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:uDGL6kOW6j0C" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-4-4285F4?logo=googlescholar&amp;labelColor=beige" alt="4 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>IEEE 802.11ah is a sub-GHz communication technology to offer longer range and low power connectivity for the Internet of Things (IoT) applications. A Restricted Access Window (RAW) is specified to decrease the collision probability. Stations are divided into groups and stations from each group attempt to access the channel by employing the Distributed Coordination Function during their assigned RAW slots. However, the network throughput is limited by a single channel MAC protocol. In this paper, Space-Frequency Diversity-based MAC protocol for the IEEE 802.11ah network (SF-MAC protocol) is proposed to allow stations of different sectors to transmit packets on different channels with the help of Forwarders. The proposed SF-MAC protocol improves the packet delivery ratio and aggregate throughput of the network.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@inproceedings</span><span class="p">{</span><span class="nl">dang2022space</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Space-Frequency Diversity based MAC protocol for IEEE 802.11 ah networks}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Dang, Duc Ngoc Minh and Tran, Van Thau and Nguyen, Hoang Lam and Pham, Nhat Truong and Tran, Anh Khoa and Dang, Ngoc-Hanh}</span><span class="p">,</span>
  <span class="na">booktitle</span> <span class="p">=</span> <span class="s">{2022 International Conference on Advanced Technologies for Communications ATC}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{159--164}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">organization</span> <span class="p">=</span> <span class="s">{IEEE}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1109/ATC55345.2022.9943042}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#215d42"> <a href="https://ieeexplore.ieee.org/xpl/conhome/1817425/all-proceedings" rel="external nofollow noopener" target="_blank">GTSD</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/KeyIE.png" sizes="200px"></source> <img src="/assets/img/publication_preview/KeyIE.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="KeyIE.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="pham2022key" class="col-sm-8"> <div class="title">Key Information Extraction from Mobile-Captured Vietnamese Receipt Images using Graph Neural Networks Approach</div> <div class="author"> Van Dung Pham , Le Quan Nguyen , <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , Bao Hung Nguyen , <a href="https://scholar.google.com/citations?hl=en&amp;user=2UKP440AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Duc Ngoc Minh Dang</a> , and <a href="https://scholar.google.com/citations?hl=en&amp;user=XFUxOkoAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Sy Dzung Nguyen</a> </div> <div class="periodical"> <em>In <i><a href="https://ieeexplore.ieee.org/xpl/conhome/1817425/all-proceedings" style="color: #00ab37; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">2022 6th International Conference on Green Technology and Sustainable Development (GTSD)</a></i></em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://ieeexplore.ieee.org/abstract/document/9989111" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://github.com/ThorPham/Key_infomation_extraction" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1109/GTSD54989.2022.9989111" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:FiytvqdAVhgC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-0-4285F4?logo=googlescholar&amp;labelColor=beige" alt="0 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Information extraction and retrieval are growing fields that have a significant role in document parser and analysis systems. Researches and applications developed in recent years show the numerous difficulties and obstacles in extracting key information from documents. Thanks to the raising of graph theory and deep learning, graph representation and graph learning have been widely applied in information extraction to obtain more exact results. In this paper, we propose a solution upon graph neural networks (GNN) for key information extraction (KIE) that aims to extract the key information from mobile-captured Vietnamese receipt images. Firstly, the images are pre-processed using U<sup>2</sup>-Net, and then a CRAFT model is used to detect texts from the pre-processed images. Next, the implemented TransformerOCR model is employed for text recognition. Finally, a GNN-based model is designed to extract the key information based on the recognized texts. For validating the effectiveness of the proposed solution, the publicly available dataset released from the Mobile-Captured Receipt Recognition (MC-OCR) Challenge 2021 is used to train and evaluate. The experimental results indicate that our proposed solution achieves a character error rate (CER) score of 0.25 on the private test set, which is more comparable with all reported solutions in the MC-OCR Challenge 2021 as mentioned in the literature. For reproducing and knowledge-sharing purposes, our implementation of the proposed solution is publicly available at <a href="https://github.com/ThorPham/Key_infomation_extraction/" rel="external nofollow noopener" target="_blank">https://github.com/ThorPham/Key_infomation_extraction/</a>.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@inproceedings</span><span class="p">{</span><span class="nl">pham2022key</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Key Information Extraction from Mobile-Captured Vietnamese Receipt Images using Graph Neural Networks Approach}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Pham, Van Dung and Nguyen, Le Quan and Pham, Nhat Truong and Nguyen, Bao Hung and Dang, Duc Ngoc Minh and Nguyen, Sy Dzung}</span><span class="p">,</span>
  <span class="na">booktitle</span> <span class="p">=</span> <span class="s">{2022 6th International Conference on Green Technology and Sustainable Development GTSD}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{232--237}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">organization</span> <span class="p">=</span> <span class="s">{IEEE}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1109/GTSD54989.2022.9989111}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#215d42"> <a href="https://ieeexplore.ieee.org/xpl/conhome/1817425/all-proceedings" rel="external nofollow noopener" target="_blank">GTSD</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/SceneText.png" sizes="200px"></source> <img src="/assets/img/publication_preview/SceneText.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="SceneText.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="pham2022vietnamese" class="col-sm-8"> <div class="title">Vietnamese Scene Text Detection and Recognition using Deep Learning: An Empirical Study</div> <div class="author"> <em><b style="color: #FFA500;">Nhat Truong Pham<sup>†</sup></b></em> , Van Dung Pham<sup>†</sup> , Qui Nguyen-Van , Bao Hung Nguyen , <a href="https://scholar.google.com/citations?hl=en&amp;user=2UKP440AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Duc Ngoc Minh Dang</a> , and <a href="https://scholar.google.com/citations?hl=en&amp;user=XFUxOkoAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Sy Dzung Nguyen</a> </div> <div class="periodical"> <em>In <i><a href="https://ieeexplore.ieee.org/xpl/conhome/1817425/all-proceedings" style="color: #00ab37; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">2022 6th International Conference on Green Technology and Sustainable Development (GTSD)</a></i></em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://ieeexplore.ieee.org/abstract/document/9989248" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://github.com/ThorPham/VN_scene_text_detection_recognition" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1109/GTSD54989.2022.9989248" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:IaI1MmNe2tcC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-5-4285F4?logo=googlescholar&amp;labelColor=beige" alt="5 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Scene text detection and recognition are vital challenging tasks in computer vision, which are to detect and recognize sequences of texts in natural scenes. Recently, researchers have investigated a lot of state-of-the-art methods to improve the accuracy and efficiency of text detection and recognition. However, there has been little research on text detection and recognition in natural scenes in Vietnam. In this paper, a deep learning-based empirical investigation of Vietnamese scene text detection and recognition is presented. Firstly, four detection models including differentiable binarization network (DBN), pyramid mask text detector (PMTD), pixel aggregation network (PAN), and Fourier contour embedding network (FCEN), are employed to detect text regions from the images. Then, four text recognition models including convolutional recurrent neural network (CRNN), self-attention text recognition network (SATRN), no-recurrence sequence-to-sequence text recognizer (NRTR), and RobustScanner (RS) are also investigated to recognize the texts. Moreover, data augmentation methods are also applied to enrich data for improving the accuracy and enhancing the performance of scene text detection and recognition. To validate the effectiveness of scene text detection and recognition models, the VinText dataset is employed for evaluation. Empirical results show that PMTD and SATRN achieve the highest scores among the others for text detection and recognition, respectively. For knowledge-sharing, our implementation is publicly available at <a href="https://github.com/ThorPham/VN_scene_text_detection_recognition/" rel="external nofollow noopener" target="_blank">https://github.com/ThorPham/VN_scene_text_detection_recognition/</a>.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@inproceedings</span><span class="p">{</span><span class="nl">pham2022vietnamese</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Vietnamese Scene Text Detection and Recognition using Deep Learning: An Empirical Study}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Pham, Nhat Truong and Pham, Van Dung and Nguyen-Van, Qui and Nguyen, Bao Hung and Dang, Duc Ngoc Minh and Nguyen, Sy Dzung}</span><span class="p">,</span>
  <span class="na">booktitle</span> <span class="p">=</span> <span class="s">{2022 6th International Conference on Green Technology and Sustainable Development GTSD}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{213--218}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">organization</span> <span class="p">=</span> <span class="s">{IEEE}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1109/GTSD54989.2022.9989248}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#215d42"> <a href="https://ieee-icce.org/" rel="external nofollow noopener" target="_blank">ICCE</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/SRE-MAC.png" sizes="200px"></source> <img src="/assets/img/publication_preview/SRE-MAC.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="SRE-MAC.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="dang2022safety" class="col-sm-8"> <div class="title">Safety Message Broadcast Reliability Enhancement MAC protocol in VANETs</div> <div class="author"> <a href="https://scholar.google.com/citations?hl=en&amp;user=2UKP440AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Duc Ngoc Minh Dang</a> , Anh Khoa Tran , <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , Khanh Duong Tran , and Hanh Ngoc Dang </div> <div class="periodical"> <em>In <i><a href="https://ieee-icce.org/" style="color: #00ab37; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">2022 IEEE Ninth International Conference on Communications and Electronics (ICCE)</a></i></em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://ieeexplore.ieee.org/abstract/document/9852052" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1109/ICCE55644.2022.9852052" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:FAceZFleit8C" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-0-4285F4?logo=googlescholar&amp;labelColor=beige" alt="0 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Recently, Vehicular Ad-hoc Networks (VANETs) have been considered as an important part of the Intelligent Transportation System. Data transmission in VANET can be safety message and non-safety message transmissions. While the safety message transmission typically requires bounded delay and a high packet delivery ratio, the non-safety message transmission demands sufficiently high throughput. In this paper, a MAC protocol for Safety message broadcast Reliability Enhancement in VANETs, named SRE-MAC protocol, is proposed to ensure both the reliability of safety message transmission and the high throughput for non-safety data transmission. In particular, the proposed SRE-MAC employs a time slot allocation of TDMA and a random-access technique of CSMA schemes for accessing the control channel. To evaluate our proposed SRE-MAC protocol, some extensive simulations are conducted. The simulation results show that the proposed SRE-MAC protocol achieves higher performance in terms of safety packet delivery ratio and throughput of non-safety packets, as compared to the IEEE 1609.4 and the VER-MAC protocol.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@inproceedings</span><span class="p">{</span><span class="nl">dang2022safety</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Safety Message Broadcast Reliability Enhancement MAC protocol in VANETs}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Dang, Duc Ngoc Minh and Tran, Anh Khoa and Pham, Nhat Truong and Tran, Khanh Duong and Dang, Hanh Ngoc}</span><span class="p">,</span>
  <span class="na">booktitle</span> <span class="p">=</span> <span class="s">{2022 IEEE Ninth International Conference on Communications and Electronics ICCE}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{69--74}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">organization</span> <span class="p">=</span> <span class="s">{IEEE}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1109/ICCE55644.2022.9852052}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://www.nature.com/srep/" rel="external nofollow noopener" target="_blank">Sci Rep</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/SciRep.png" sizes="200px"></source> <img src="/assets/img/publication_preview/SciRep.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="SciRep.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="tran2022deep" class="col-sm-8"> <div class="title">A deep learning approach for detecting drill bit failures from a small sound dataset</div> <div class="author"> Thanh Tran , <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , and Jan Lundgren </div> <div class="periodical"> <em><i><a href="https://www.nature.com/srep/" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Scientific Reports</a></i></em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://www.nature.com/articles/s41598-022-13237-7" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/DL_drill_bit_failure_detection.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a> <a href="https://github.com/thanhtran1965/DrillFailureDetection_SciRep2022" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1038/s41598-022-13237-7" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:zGdJYJv2LkUC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-17-4285F4?logo=googlescholar&amp;labelColor=beige" alt="17 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Monitoring the conditions of machines is vital in the manufacturing industry. Early detection of faulty components in machines for stopping and repairing the failed components can minimize the downtime of the machine. In this article, we present a method for detecting failures in drill machines using drill sounds in Valmet AB, a company in Sundsvall, Sweden that supplies equipment and processes for the production of pulp, paper, and biofuels. The drill dataset includes two classes: anomalous sounds and normal sounds. Detecting drill failure effectively remains a challenge due to the following reasons. The waveform of drill sound is complex and short for detection. Furthermore, in realistic soundscapes, both sounds and noise exist simultaneously. Besides, the balanced dataset is small to apply state-of-the-art deep learning techniques. Due to these aforementioned difficulties, sound augmentation methods were applied to increase the number of sounds in the dataset. In this study, a convolutional neural network (CNN) was combined with a long-short-term memory (LSTM) to extract features from log-Mel spectrograms and to learn global representations of two classes. A leaky rectified linear unit (Leaky ReLU) was utilized as the activation function for the proposed CNN instead of the ReLU. Moreover, an attention mechanism was deployed at the frame level after the LSTM layer to pay attention to the anomaly in sounds. As a result, the proposed method reached an overall accuracy of 92.62% to classify two classes of machine sounds on Valmet’s dataset. In addition, an extensive experiment on another drilling dataset with short sounds yielded 97.47% accuracy. With multiple classes and long-duration sounds, an experiment utilizing the publicly available UrbanSound8K dataset obtains 91.45%. Extensive experiments on our dataset as well as publicly available datasets confirm the efficacy and robustness of our proposed method. For reproducing and deploying the proposed system, an open-source repository is publicly available at <a href="https://github.com/thanhtran1965/DrillFailureDetection_SciRep2022/" rel="external nofollow noopener" target="_blank">https://github.com/thanhtran1965/DrillFailureDetection_SciRep2022/</a>.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">tran2022deep</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{A deep learning approach for detecting drill bit failures from a small sound dataset}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Tran, Thanh and Pham, Nhat Truong and Lundgren, Jan}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Scientific Reports}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{12}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{1}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{9623}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Nature Publishing Group UK London}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1038/s41598-022-13237-7}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://onlinelibrary.wiley.com/journal/1096987x" rel="external nofollow noopener" target="_blank">JCC</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/mVina.png" sizes="200px"></source> <img src="/assets/img/publication_preview/mVina.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="mVina.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="pham2022improving" class="col-sm-8"> <div class="title">Improving ligand-ranking of AutoDock Vina by changing the empirical parameters</div> <div class="author"> <a href="https://scholar.google.com/citations?hl=en&amp;user=_aQ5P4gAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">T Ngoc Han Pham</a> , <a href="https://scholar.google.com/citations?hl=en&amp;user=a4xyHScAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Trung Hai Nguyen</a> , Nguyen Minh Tam , Thien Y. Vu , <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , Nguyen Truong Huy , Binh Khanh Mai , Nguyen Thanh Tung , Minh Quan Pham , Van V. Vu , and <a href="https://scholar.google.com/citations?hl=en&amp;user=_VxSvQkAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Son Tung Ngo</a> </div> <div class="periodical"> <em><i><a href="https://onlinelibrary.wiley.com/journal/1096987x" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Journal of Computational Chemistry</a></i></em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/jcc.26779" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://github.com/nhattruongpham/mvina" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1002/jcc.26779" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:t6usbXjVLHcC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-41-4285F4?logo=googlescholar&amp;labelColor=beige" alt="41 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>AutoDock Vina (Vina) achieved a very high docking-success rate, <i>p̂</i>, but give a rather low correlation coefficient, <i>R</i>, for binding affinity with respect to experiments. This low correlation can be an obstacle for ranking of ligand-binding affinity, which is the main objective of docking simulations. In this context, we evaluated the dependence of Vina <i>R</i> coefficient upon its empirical parameters. <i>R</i> is affected more by changing the gauss2 and rotation than other terms. The docking-success rate <i>p̂</i> is sensitive to the alterations of the gauss1, gauss2, repulsion, and hydrogen bond parameters. Based on our benchmarks, the parameter set1 has been suggested to be the most optimal. The testing study over 800 complexes indicated that the modified Vina provided higher correlation with experiment <i>R<sub>set1</sub>=0.556±0.025</i> compared with <i>R<sub>Default</sub>=0.493±0.028</i> obtained by the original Vina and <i>R<sub>Vina 1.2</sub>=0.503±0.029</i> by Vina version 1.2. Besides, the modified Vina can be also applied more widely, giving <i>R ≥ 0.500</i> for 32/48 targets, compared with the default package, giving <i>R ≥ 0.500</i> for 31/48 targets. In addition, validation calculations for 1036 complexes obtained from version 2019 of PDBbind refined structures showed that the set1 of parameters gave higher correlation coefficient (<i>R<sub>set1</sub>=0.617±0.017</i>) than the default package (<i>R<sub>Default</sub>=0.543±0.020</i>) and Vina version 1.2 (<i>R<sub>Vina 1.2</sub>=0.540±0.020</i>). The version of Vina with set1 of parameters can be downloaded at <a href="https://github.com/sontungngo/mvina/" rel="external nofollow noopener" target="_blank">https://github.com/sontungngo/mvina/</a>. The outcomes would enhance the ranking of ligand-binding affinity using Autodock Vina.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">pham2022improving</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Improving ligand-ranking of AutoDock Vina by changing the empirical parameters}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Pham, T Ngoc Han and Nguyen, Trung Hai and Tam, Nguyen Minh and Y. Vu, Thien and Pham, Nhat Truong and Huy, Nguyen Truong and Mai, Binh Khanh and Tung, Nguyen Thanh and Pham, Minh Quan and V. Vu, Van and Ngo, Son Tung}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Journal of Computational Chemistry}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{43}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{3}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{160--169}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{Wiley Online Library}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1002/jcc.26779}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://cis.ieee.org/publications/t-fuzzy-systems" rel="external nofollow noopener" target="_blank">IEEE TFS</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/fRiskC.png" sizes="200px"></source> <img src="/assets/img/publication_preview/fRiskC.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="fRiskC.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="nguyen2022determination" class="col-sm-8"> <div class="title">Determination of the optimal number of clusters: a fuzzy-set based method</div> <div class="author"> <a href="https://scholar.google.com/citations?hl=en&amp;user=XFUxOkoAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Sy Dzung Nguyen</a> , Vu Song Thuy Nguyen , and <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> </div> <div class="periodical"> <em><i><a href="https://cis.ieee.org/publications/t-fuzzy-systems" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">IEEE Transactions on Fuzzy Systems</a></i></em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://ieeexplore.ieee.org/abstract/document/9562269" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1109/TFUZZ.2021.3118113" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:pyW8ca7W8N0C" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-22-4285F4?logo=googlescholar&amp;labelColor=beige" alt="22 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>The optimal number of clusters (<i>C<sub>opt</sub></i>) is one of the determinants of clustering efficiency. In this article, we present a new method of quantifying <i>C<sub>opt</sub></i> for centroid-based clustering. First, we propose a new clustering validity index named fRisk(<i>C</i>) based on the fuzzy set theory. It takes the role of normalization and accumulation of local risks coming from each action either splitting data from a cluster or merging data into a cluster. fRisk(<i>C</i>) exploits the local distribution information of the database to catch the global information of the clustering process in the form of the risk degree. Based on the monotonous reduction property of fRisk(<i>C</i>), which is proved theoretically, we present a fRisk-based new algorithm named fRisk4-bA for determining <i>C<sub>opt</sub></i>. In the algorithm, the well-known L-method is employed as a supplemented tool to catch <i>C<sub>opt</sub></i> on the graph of the fRisk(<i>C</i>). Along with the stable convergence trend of the method to be proved theoretically, numerical surveys are also carried out. The surveys show that the high reliability and stability, as well as the sensitivity in separating/merging clusters in high-density areas, even if the presence of noise in the databases, are the strong points of the proposed method.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">nguyen2022determination</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Determination of the optimal number of clusters: a fuzzy-set based method}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Nguyen, Sy Dzung and Nguyen, Vu Song Thuy and Pham, Nhat Truong}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{IEEE Transactions on Fuzzy Systems}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{30}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{9}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{3514--3526}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
  <span class="na">publisher</span> <span class="p">=</span> <span class="s">{IEEE}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1109/TFUZZ.2021.3118113}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> <li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#215d42"> <a href="https://defactify.com/" rel="external nofollow noopener" target="_blank">DE-FACTIFY@AAAI</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/Memotion2.png" sizes="200px"></source> <img src="/assets/img/publication_preview/Memotion2.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="Memotion2.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="nguyen2022hcilab" class="col-sm-8"> <div class="title"> <b><i>HCILab at Memotion 2.0 2022:</i></b> Analysis of sentiment, emotion and intensity of emotion classes from meme images using single and multi modalities</div> <div class="author"> <a href="https://scholar.google.com/citations?hl=en&amp;user=zSAfD80AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Thanh Tin Nguyen</a> , <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , <a href="https://scholar.google.com/citations?hl=en&amp;user=At-Y-H8AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Ngoc Duy Nguyen</a> , <a href="https://scholar.google.com/citations?hl=en&amp;user=5b9ncWoAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Hai Nguyen</a> , Long H. Nguyen , and <a href="https://scholar.google.com/citations?hl=en&amp;user=gGfbXFIAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Yong-Guk Kim</a> </div> <div class="periodical"> <em>In <i><a href="https://defactify.com/" style="color: #00ab37; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">DE-FACTIFY@ AAAI</a></i></em>, 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://ceur-ws.org/Vol-3199/" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/HCILab_Memotion2.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a> <a href="https://github.com/ngthanhtin/Memotion2_AAAI_WS_2022" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-pmid="" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:_FM0Bhl9EiAC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-10-4285F4?logo=googlescholar&amp;labelColor=beige" alt="10 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Nowadays, memes found on internet are overwhelming. Although they are innocuous and sometimes entertaining, there exist memes that contain sarcasm, offensive, or motivational feelings. In this study, several approaches are proposed to solve the multiple modality problem in analysing the given meme dataset. The imbalance issue has been addressed by using a new Auto Augmentation method and the uncorrelation issue has been mitigated by adopting deep Canonical Correlation Analysis to find the most correlated projections of visual and textual feature embedding. In addition, both stacked attention and multi-hop attention network are employed to efficiently generate aggregated features. As a result, our team, i.e. HCILab, achieved a weighted F1 score of 0.4995 for sentiment analysis, 0.7414 for emotion classification, and 0.5301 for scale/intensity of emotion classes on the leaderboard. This results are obtained by using concatenation between image and text model and our code can be found at <a href="https://github.com/ngthanhtin/Memotion2_AAAI_WS_2022/" rel="external nofollow noopener" target="_blank">https://github.com/ngthanhtin/Memotion2_AAAI_WS_2022/</a>.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@inproceedings</span><span class="p">{</span><span class="nl">nguyen2022hcilab</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{HCILab at Memotion 2.0 2022: Analysis of sentiment, emotion and intensity of emotion classes from meme images using single and multi modalities}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Nguyen, Thanh Tin and Pham, Nhat Truong and Nguyen, Ngoc Duy and Nguyen, Hai and Nguyen, Long H. and Kim, Yong-Guk}</span><span class="p">,</span>
  <span class="na">booktitle</span> <span class="p">=</span> <span class="s">{DE-FACTIFY&amp;#64; AAAI}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2022}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li> </ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#215d42"> <a href="https://ieeexplore.ieee.org/xpl/conhome/1811304/all-proceedings" rel="external nofollow noopener" target="_blank">SSCI</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/soundSepsound.png" sizes="200px"></source> <img src="/assets/img/publication_preview/soundSepsound.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="soundSepsound.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="tran2021separate" class="col-sm-8"> <div class="title">Separate sound into STFT frames to eliminate sound noise frames in sound classification</div> <div class="author"> Thanh Tran , Kien Bui Huy , <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , Marco Carratù , Consolatina Liguori , and Jan Lundgren </div> <div class="periodical"> <em>In <i><a href="https://ieeexplore.ieee.org/xpl/conhome/1811304/all-proceedings" style="color: #00ab37; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">2021 IEEE Symposium Series on Computational Intelligence (SSCI)</a></i></em>, 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://ieeexplore.ieee.org/abstract/document/9660125" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="https://github.com/nhattruongpham/soundSepsound" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">Code</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.1109/SSCI50451.2021.9660125" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:V3AGJWp-ZtQC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-5-4285F4?logo=googlescholar&amp;labelColor=beige" alt="5 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Sounds always contain acoustic noise and background noise that affects the accuracy of the sound classification system. Hence, suppression of noise in the sound can improve the robustness of the sound classification model. This paper investigated a sound separation technique that separates the input sound into many overlapped-content Short-Time Fourier Transform (STFT) frames. Our approach is different from the traditional STFT conversion method, which converts each sound into a single STFT image. Contradictory, separating the sound into many STFT frames improves model prediction accuracy by increasing variability in the data and therefore learning from that variability. These separated frames are saved as images and then labeled manually as clean and noisy frames which are then fed into transfer learning convolutional neural networks (CNNs) for the classification task. The pre-trained CNN architectures that learn from these frames become robust against the noise. The experimental results show that the proposed approach is robust against noise and achieves 94.14% in terms of classifying 21 classes including 20 classes of sound events and a noisy class. An open-source repository of the proposed method and results is available at <a href="https://github.com/nhattruongpham/soundSepsound/" rel="external nofollow noopener" target="_blank">https://github.com/nhattruongpham/soundSepsound/</a>.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@inproceedings</span><span class="p">{</span><span class="nl">tran2021separate</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{Separate sound into STFT frames to eliminate sound noise frames in sound classification}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Tran, Thanh and Huy, Kien Bui and Pham, Nhat Truong and Carrat{\`u}, Marco and Liguori, Consolatina and Lundgren, Jan}</span><span class="p">,</span>
  <span class="na">booktitle</span> <span class="p">=</span> <span class="s">{2021 IEEE Symposium Series on Computational Intelligence SSCI}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{1--7}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2021}</span><span class="p">,</span>
  <span class="na">organization</span> <span class="p">=</span> <span class="s">{IEEE}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.1109/SSCI50451.2021.9660125}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> <h2 class="bibliography">2020</h2> <ol class="bibliography"><li> <div class="row"> <div class="col col-sm-2 abbr"> <abbr class="badge rounded w-100" style="background-color:#600"> <a href="https://jaec.vn/index.php/JAEC" rel="external nofollow noopener" target="_blank">JAEC</a> </abbr> <figure> <picture> <source class="responsive-img-srcset" srcset="/assets/img/publication_preview/JAEC.png" sizes="200px"></source> <img src="/assets/img/publication_preview/JAEC.png" class="preview z-depth-1 rounded" width="100%" height="auto" alt="JAEC.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="pham2020method" class="col-sm-8"> <div class="title">A method upon deep learning for speech emotion recognition</div> <div class="author"> <em><b style="color: #FFA500;">Nhat Truong Pham</b></em> , <a href="https://scholar.google.com/citations?hl=en&amp;user=2UKP440AAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Duc Ngoc Minh Dang</a> , and <a href="https://scholar.google.com/citations?hl=en&amp;user=XFUxOkoAAAAJ&amp;view_op=list_works&amp;sortby=pubdate" rel="external nofollow noopener" target="_blank">Sy Dzung Nguyen</a> </div> <div class="periodical"> <em><i><a href="https://jaec.vn/index.php/JAEC" style="color: #FF3636; text-decoration: underline dashed;" rel="external nofollow noopener" target="_blank">Journal of Advanced Engineering and Computation</a></i></em>, 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a class="bibtex btn btn-sm z-depth-0" role="button">Bib</a> <a href="https://jaec.vn/index.php/JAEC/article/view/311" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/ADCRNN.pdf" class="btn btn-sm z-depth-0" role="button" target="_blank">PDF</a> </div> <div class="badges"> <span class="__dimensions_badge_embed__" data-doi="10.25073/jaec.202044.311" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=HybH2XkAAAAJ&amp;citation_for_view=HybH2XkAAAAJ:GnPB-g6toBAC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-12-4285F4?logo=googlescholar&amp;labelColor=beige" alt="12 Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>Feature extraction and emotional classification are significant roles in speech emotion recognition. It is hard to extract and select the optimal features, researchers can not be sure what the features should be. With deep learning approaches, features could be extracted by using hierarchical abstraction layers, but it requires high computational resources and a large number of data. In this article, we choose static, differential, and acceleration coefficients of log Mel-spectrogram as inputs for the deep learning model. To avoid performance degradation, we also add a skip connection with dilated convolution network integration. All representatives are fed into a self-attention mechanism with bidirectional recurrent neural networks to learn long term global features and exploit context for each time step. Finally, we investigate contrastive center loss with softmax loss as loss function to improve the accuracy of emotion recognition. For validating robustness and effectiveness, we tested the proposed method on the Emo-DB and ERC2019 datasets. Experimental results show that the performance of the proposed method is strongly comparable with the existing state-of-the-art methods on the Emo-DB and ERC2019 with 88% and 67%, respectively.</p> </div> <div class="bibtex hidden"> <figure class="highlight"><pre><code class="language-bibtex" data-lang="bibtex"><span class="nc">@article</span><span class="p">{</span><span class="nl">pham2020method</span><span class="p">,</span>
  <span class="na">title</span> <span class="p">=</span> <span class="s">{A method upon deep learning for speech emotion recognition}</span><span class="p">,</span>
  <span class="na">author</span> <span class="p">=</span> <span class="s">{Pham, Nhat Truong and Dang, Duc Ngoc Minh and Nguyen, Sy Dzung}</span><span class="p">,</span>
  <span class="na">journal</span> <span class="p">=</span> <span class="s">{Journal of Advanced Engineering and Computation}</span><span class="p">,</span>
  <span class="na">volume</span> <span class="p">=</span> <span class="s">{4}</span><span class="p">,</span>
  <span class="na">number</span> <span class="p">=</span> <span class="s">{4}</span><span class="p">,</span>
  <span class="na">pages</span> <span class="p">=</span> <span class="s">{273--285}</span><span class="p">,</span>
  <span class="na">year</span> <span class="p">=</span> <span class="s">{2020}</span><span class="p">,</span>
  <span class="na">doi</span> <span class="p">=</span> <span class="s">{10.25073/jaec.202044.311}</span><span class="p">,</span>
<span class="p">}</span></code></pre></figure> </div> </div> </div> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> © Copyright 2023 - 2025 Nhat Truong Pham. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. This work is licensed under <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/?ref=chooser-v1" target="_blank" rel="license noopener noreferrer" style="display:inline-block;">CC BY-NC-ND 4.0<img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;" src="https://mirrors.creativecommons.org/presskit/icons/cc.svg?ref=chooser-v1" alt=""><img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;" src="https://mirrors.creativecommons.org/presskit/icons/by.svg?ref=chooser-v1" alt=""><img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;" src="https://mirrors.creativecommons.org/presskit/icons/nc.svg?ref=chooser-v1" alt=""><img style="height:22px!important;margin-left:3px;vertical-align:text-bottom;" src="https://mirrors.creativecommons.org/presskit/icons/nd.svg?ref=chooser-v1" alt=""></a> </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?e0514a05c5c95ac1a93a8dfd5249b92e"></script> <script defer src="/assets/js/copy_code.js?12775fdf7f95e901d7119054556e495f" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>addBackToTop();</script> <script type="module" src="/assets/js/search/ninja-keys.min.js?a3446f084dcaecc5f75aa1757d087dcf"></script> <ninja-keys hidebreadcrumbs noautoloadmdicons placeholder="Type to start searching"></ninja-keys> <script>let searchTheme=determineComputedTheme();const ninjaKeys=document.querySelector("ninja-keys");"dark"===searchTheme?ninjaKeys.classList.add("dark"):ninjaKeys.classList.remove("dark");const openSearchModal=()=>{const e=$("#navbarNav");e.hasClass("show")&&e.collapse("hide"),ninjaKeys.open()};</script> <script>const ninja=document.querySelector("ninja-keys");ninja.data=[{id:"nav-home",title:"Home",section:"Navigation",handler:()=>{window.location.href="/"}},{id:"nav-news",title:"News",description:"",section:"Navigation",handler:()=>{window.location.href="/news/"}},{id:"nav-publications",title:"Publications",description:"",section:"Navigation",handler:()=>{window.location.href="/publications/"}},{id:"nav-collaborators",title:"Collaborators",description:"",section:"Navigation",handler:()=>{window.location.href="/collaborators/"}},{id:"nav-mentees",title:"Mentees",description:"",section:"Navigation",handler:()=>{window.location.href="/mentees/"}},{id:"nav-repositories",title:"Repositories",description:"",section:"Navigation",handler:()=>{window.location.href="/repositories/"}},{id:"nav-cv",title:"CV",description:"",section:"Navigation",handler:()=>{window.location.href="/cv/"}},{id:"dropdown-molecules",title:"Molecules",description:"",section:"Dropdown",handler:()=>{window.location.href=""}},{id:"dropdown-dna",title:"DNA",description:"",section:"Dropdown",handler:()=>{window.location.href=""}},{id:"dropdown-rna",title:"RNA",description:"",section:"Dropdown",handler:()=>{window.location.href=""}},{id:"dropdown-peptide",title:"Peptide",description:"",section:"Dropdown",handler:()=>{window.location.href=""}},{id:"dropdown-protein",title:"Protein",description:"",section:"Dropdown",handler:()=>{window.location.href=""}},{id:"dropdown-microbiome",title:"Microbiome",description:"",section:"Dropdown",handler:()=>{window.location.href=""}},{id:"dropdown-imaging",title:"Imaging",description:"",section:"Dropdown",handler:()=>{window.location.href=""}},{id:"dropdown-skku-rs-24",title:"SKKU-RS-24",description:"",section:"Dropdown",handler:()=>{window.location.href=""}},{id:"dropdown-ijcai-24",title:"IJCAI-24",description:"",section:"Dropdown",handler:()=>{window.location.href=""}},{id:"dropdown-acl-24",title:"ACL-24",description:"",section:"Dropdown",handler:()=>{window.location.href=""}},{id:"news-one-collaborative-manuscript-entitled-drugormerdti-drug-graphormer-for-drug-target-interaction-prediction-has-been-accepted-for-publication-in-the-computers-in-biology-and-medicine-journal",title:"One collaborative manuscript entitled \u201cDrugormerDTI: Drug Graphormer for drug\u2013target interaction prediction\u201d has been...",description:"",section:"News"},{id:"news-one-manuscript-entitled-an-exploratory-simulation-study-and-prediction-model-on-human-brain-behavior-and-activity-using-an-integration-of-deep-neural-network-and-biosensor-rabi-antenna-has-been-accepted-for-publication-in-the-heliyon-journal",title:"One manuscript entitled \u201cAn exploratory simulation study and prediction model on human brain...",description:"",section:"News"},{id:"news-one-collaborative-manuscript-entitled-towards-an-efficient-machine-learning-model-for-financial-time-series-forecasting-has-been-accepted-for-publication-in-the-soft-computing-journal",title:"One collaborative manuscript entitled \u201cTowards an efficient machine learning model for financial time...",description:"",section:"News"},{id:"news-one-manuscript-entitled-hybrid-data-augmentation-and-deep-attention-based-dilated-convolutional-recurrent-neural-networks-for-speech-emotion-recognition-has-been-accepted-for-publication-in-the-expert-systems-with-applications-journal",title:"One manuscript entitled \u201cHybrid data augmentation and deep attention-based dilated convolutional-recurrent neural networks...",description:"",section:"News"},{id:"news-released-new-officially-personal-website",title:"Released new officially personal website",description:"",section:"News"},{id:"news-one-collaborative-manuscript-entitled-adp-fuse-a-novel-two-layer-machine-learning-predictor-to-identify-antidiabetic-peptides-and-diabetes-types-using-multiview-information-has-been-accepted-for-publication-in-the-computers-in-biology-and-medicine-journal",title:"One collaborative manuscript entitled \u201cADP-Fuse: A novel two-layer machine learning predictor to identify...",description:"",section:"News"},{id:"news-one-collaborative-manuscript-entitled-comparative-analysis-of-multi-loss-functions-for-enhanced-multi-modal-speech-emotion-recognition-has-been-accepted-for-publication-in-the-2023-14th-international-conference-on-information-and-communication-technology-convergence-ictc",title:"One collaborative manuscript entitled \u201cComparative analysis of multi-loss functions for enhanced multi-modal speech...",description:"",section:"News"},{id:"news-one-manuscript-entitled-ser-fuse-an-emotion-recognition-application-utilizing-multi-modal-multi-lingual-and-multi-feature-fusion-has-been-accepted-for-publication-in-the-proceedings-of-the-12th-international-symposium-on-information-and-communication-technology",title:"One manuscript entitled \u201cSER-Fuse: An Emotion Recognition Application Utilizing Multi-Modal, Multi-Lingual, and Multi-Feature...",description:"",section:"News"},{id:"news-one-of-my-very-first-first-author-manuscripts-in-the-field-of-computational-biology-and-bioinformatics-entitled-advancing-the-accuracy-of-sars-cov-2-phosphorylation-site-detection-via-meta-learning-approach-has-been-accepted-for-publication-in-the-briefings-in-bioinformatics-journal",title:"One of my very first first-author manuscripts in the field of Computational Biology...",description:"",section:"News"},{id:"news-our-project-entitled-identification-of-human-2-o-methylation-2om-sites-using-a-hybrid-deep-learning-framework-\ud558\uc774\ube0c\ub9ac\ub4dc-\ub525-\ub7ec\ub2dd-\ud504\ub808\uc784\uc6cc\ud06c\ub97c-\uc0ac\uc6a9\ud55c-\uc778\uac04-2-o-\uba54\ud2f8\ud654-2om-\ubd80\uc704-\uc2dd\ubcc4-has-been-selected-for-the-2023-2nd-k-bds-analysis-infrastructure-utilization-support-program-track-i-large-innovation-research-by-the-korea-bio-data-station-k-bds-korea-institute-of-science-and-technology-information-republic-of-korea",title:"Our project, entitled \u201cIdentification of human 2\u2019-O-methylation (2OM) sites using a hybrid deep...",description:"",section:"News"},{id:"news-two-collaborative-manuscripts-entitled-deep-learning-based-automated-cashier-system-for-bakeries-and-innovative-multi-modal-control-for-surveillance-spider-robot-an-integration-of-voice-and-hand-gesture-recognition-have-been-accepted-for-publication-in-proceedings-of-the-2024-9th-international-conference-on-intelligent-information-technology",title:"Two collaborative manuscripts entitled \u201cDeep Learning-Based Automated Cashier System for Bakeries\u201d and \u201cInnovative...",description:"",section:"News"},{id:"news-one-collaborative-manuscript-entitled-enhanced-sliding-mode-controller-design-via-meta-heuristic-algorithm-for-robust-and-stable-load-frequency-control-in-multi-area-power-systems-has-been-accepted-for-publication-in-the-iet-generation-transmission-amp-amp-distribution-journal",title:"One collaborative manuscript entitled \u201cEnhanced sliding mode controller design via meta-heuristic algorithm for...",description:"",section:"News"},{id:"news-one-manuscript-entitled-h2opred-a-robust-and-efficient-hybrid-deep-learning-model-for-predicting-2-o-methylation-sites-in-human-rna-has-been-accepted-for-publication-in-the-briefings-in-bioinformatics-journal",title:"One manuscript entitled \u201cH2Opred: a robust and efficient hybrid deep learning model for...",description:"",section:"News"},{id:"news-our-paper-entitled-advancing-the-accuracy-of-sars-cov-2-phosphorylation-site-detection-via-meta-learning-approach-recently-published-in-the-briefings-in-bioinformatics-journal-has-been-cataloged-in-hanbitsa-paper-of-the-biological-research-information-center-bric",title:"Our paper, entitled \u201cAdvancing the accuracy of SARS-CoV-2 phosphorylation site detection via meta-learning...",description:"",section:"News"},{id:"news-my-hanbitsa-interview-regarding-the-recently-published-paper-in-the-briefings-in-bioinformatics-journal-has-been-shared-on-the-biological-research-information-center-bric",title:"My Hanbitsa Interview regarding the recently published paper in the Briefings in Bioinformatics...",description:"",section:"News"},{id:"news-our-paper-entitled-h2opred-a-robust-and-efficient-hybrid-deep-learning-model-for-predicting-2-o-methylation-sites-in-human-rna-recently-published-in-the-briefings-in-bioinformatics-journal-has-been-cataloged-in-hanbitsa-paper-of-the-biological-research-information-center-bric",title:"Our paper, entitled \u201cH2Opred: a robust and efficient hybrid deep learning model for...",description:"",section:"News"},{id:"news-our-research-story-kor-eng-regarding-the-recently-published-papers-in-the-briefings-in-bioinformatics-journal-has-been-highlighted-on-skku-s-research-stories-kor-eng",title:"Our Research Story (KOR/ENG) regarding the recently published papers in the Briefings in...",description:"",section:"News"},{id:"news-our-project-entitled-enhancing-peptide-hla-class-i-binding-prediction-through-siamese-network-based-contrastive-learning-framework-with-feature-fusion-\ud3a9\ud0c0\uc774\ub4dc-hla-class-i-\uacb0\ud569-\uc608\uce21-\uac15\ud654\ub97c-\uc704\ud55c-\ud2b9\uc9d5-\uc735\ud569-\uae30\ubc18-\uc2dc\uc554-\ub124\ud2b8\uc6cc\ud06c-\ub300\uc870-\ud559\uc2b5-\ud504\ub808\uc784\uc6cc\ud06c-has-been-selected-for-the-2024-1st-k-bds-analysis-infrastructure-utilization-support-program-track-i-large-innovation-research-by-the-korea-bio-data-station-k-bds-korea-institute-of-science-and-technology-information-republic-of-korea",title:"Our project, entitled \u201cEnhancing Peptide-HLA Class I Binding Prediction through Siamese Network-Based Contrastive...",description:"",section:"News"},{id:"news-our-project-entitled-enhancing-m6a-rna-modification-prediction-across-cell-lines-and-tissues-using-biological-language-models-and-a-contrastive-learning-framework-has-been-selected-for-the-high-performance-computing-support-project-by-the-korea-association-for-ict-promotion-kait-republic-of-korea",title:"Our project, entitled \u201cEnhancing m6A RNA modification prediction across cell lines and tissues...",description:"",section:"News"},{id:"news-one-manuscript-entitled-ac4c-afl-a-high-precision-identification-of-human-mrna-n4-acetylcytidine-sites-based-on-adaptive-feature-representation-learning-has-been-accepted-for-publication-in-the-molecular-therapy-nucleic-acids-journal",title:"One manuscript entitled \u201cac4C-AFL: A high-precision identification of human mRNA N4-acetylcytidine sites based...",description:"",section:"News"},{id:"news-one-collaborative-manuscript-entitled-meta-2om-a-multi-classifier-meta-model-for-the-accurate-prediction-of-rna-2-o-methylation-sites-in-human-rna-has-been-accepted-for-publication-in-the-plos-one-journal",title:"One collaborative manuscript entitled \u201cMeta-2OM: A multi-classifier meta-model for the accurate prediction of...",description:"",section:"News"},{id:"news-one-collaborative-manuscript-entitled-predicting-drought-stress-under-climate-change-in-the-southern-central-highlands-of-vietnam-has-been-accepted-for-publication-in-the-environmental-monitoring-and-assessment-journal",title:"One collaborative manuscript entitled \u201cPredicting drought stress under climate change in the Southern...",description:"",section:"News"},{id:"news-one-collaborative-manuscript-entitled-sep-algpro-an-efficient-allergen-prediction-tool-utilizing-traditional-machine-learning-and-deep-learning-techniques-with-protein-language-model-features-has-been-accepted-for-publication-in-the-international-journal-of-biological-macromolecules",title:"One collaborative manuscript entitled \u201cSEP-AlgPro: An efficient allergen prediction tool utilizing traditional machine...",description:"",section:"News"},{id:"news-one-manuscript-entitled-macppred-2-0-stacked-deep-learning-for-anticancer-peptide-prediction-with-integrated-spatial-and-probabilistic-feature-representations-has-been-accepted-for-publication-in-the-journal-of-molecular-biology",title:"One manuscript entitled \u201cmACPpred 2.0: Stacked Deep Learning for Anticancer Peptide Prediction with...",description:"",section:"News"},{id:"news-our-project-entitled-predicting-twelve-common-rna-modification-sites-using-advanced-deep-learning-frameworks-with-conventional-and-pre-trained-language-model-features-\uae30\uc874-\ubc0f-\uc0ac\uc804-\ud559\uc2b5\ub41c-\uc5b8\uc5b4-\ubaa8\ub378-\uae30\ubc18-\ud2b9\uc9d5\uc744-\uc0ac\uc6a9\ud558\uc5ec-\uace0\uae09-\ub525\ub7ec\ub2dd-\ud504\ub808\uc784\uc6cc-\ud06c\ub85c-\uc5f4\ub450-\uac1c\uc758-\uc77c\ubc18\uc801\uc778-rna-\uc218\uc815-\ubd80\uc704-\uc608\uce21-has-been-selected-for-the-2024-2nd-k-bds-analysis-infrastructure-utilization-support-program-track-i-large-innovation-research-by-the-korea-bio-data-station-k-bds-korea-institute-of-science-and-technology-information-republic-of-korea",title:"Our project, entitled \u201cPredicting Twelve Common RNA Modification Sites Using Advanced Deep Learning...",description:"",section:"News"},{id:"news-one-manuscript-entitled-hotgpred-enhancing-human-o-linked-threonine-glycosylation-prediction-using-integrated-pretrained-protein-language-model-based-features-and-multi-stage-feature-selection-approach-has-been-accepted-for-publication-in-the-computers-in-biology-and-medicine-journal",title:"One manuscript entitled \u201cHOTGpred: Enhancing human O-linked threonine glycosylation prediction using integrated pretrained...",description:"",section:"News"},{id:"news-two-manuscripts-entitled-mol2lang-vlm-vision-and-text-guided-generative-pre-trained-language-models-for-advancing-molecule-captioning-through-multimodal-fusion-and-lang2mol-diff-a-diffusion-based-generative-model-for-language-to-molecule-translation-leveraging-selfies-representation-have-been-accepted-for-publication-in-the-proceedings-of-the-1st-workshop-on-language-molecules-l-m-2024",title:"Two manuscripts entitled \u201cMol2Lang-VLM: Vision- and Text-Guided Generative Pre-trained Language Models for Advancing...",description:"",section:"News"},{id:"news-two-collaborative-manuscripts-entitled-federated-learning-with-u-net-for-brain-tumor-segmentation-impact-of-client-numbers-and-data-distribution-and-towards-real-time-vietnamese-traffic-sign-recognition-on-embedded-systems-have-been-accepted-for-publication-in-the-2024-15th-international-conference-on-information-and-communication-technology-convergence-ictc",title:"Two collaborative manuscripts entitled \u201cFederated Learning with U-Net for Brain Tumor Segmentation: Impact...",description:"",section:"News"},{id:"news-our-project-entitled-accurate-liver-pathology-identification-via-dual-stream-graph-convolutional-networks-\uc774\uc911-\uc2a4\ud2b8\ub9bc-\uadf8\ub798\ud504-\ud569\uc131\uacf1-\uc2e0\uacbd\ub9dd\uc744-\ud1b5\ud55c-\uc815\ud655\ud55c-\uac04-\ubcd1\ub9ac-\uc2dd\ubcc4-has-been-selected-for-the-2024-3rd-k-bds-analysis-infrastructure-utilization-support-program-track-i-large-innovation-research-by-the-korea-bio-data-station-k-bds-korea-institute-of-science-and-technology-information-republic-of-korea",title:"Our project, entitled \u201cAccurate Liver Pathology Identification via Dual-stream Graph Convolutional Networks (\uc774\uc911...",description:"",section:"News"},{id:"news-one-collaborative-manuscript-entitled-hubert-clap-contrastive-learning-based-multimodal-emotion-recognition-using-self-alignment-approach-has-been-accepted-for-publication-in-the-mmasia-24-proceedings-of-the-6th-acm-international-conference-on-multimedia-in-asia",title:"One collaborative manuscript entitled \u201cHuBERT-CLAP: Contrastive Learning-Based Multimodal Emotion Recognition using Self-Alignment Approach\u201d...",description:"",section:"News"},{id:"news-one-manuscript-entitled-mst-m6a-a-novel-multi-scale-transformer-based-framework-for-accurate-prediction-of-m6a-modification-sites-across-diverse-cellular-contexts-has-been-accepted-for-publication-in-the-journal-of-molecular-biology",title:"One manuscript entitled \u201cMST-m6A: A Novel Multi-Scale Transformer-based Framework for Accurate Prediction of...",description:"",section:"News"},{id:"news-successfully-completed-the-preliminary-defense-for-my-ph-d-dissertation-and-became-a-ph-d-candidate",title:"Successfully completed the preliminary defense for my Ph.D. dissertation and became a Ph.D....",description:"",section:"News"},{id:"news-one-manuscript-entitled-leveraging-deep-transfer-learning-and-explainable-ai-for-accurate-covid-19-diagnosis-insights-from-a-multi-national-chest-ct-scan-study-has-been-accepted-for-publication-in-the-computers-in-biology-and-medicine-journal",title:"One manuscript entitled \u201cLeveraging deep transfer learning and explainable AI for accurate COVID-19...",description:"",section:"News"},{id:"news-received-the-skku-bk21-innovative-research-scholarship-\uad50\uc721\uc5f0\uad6c\ub2e8-\uc7a5\ud559\ud601\uc2e0-skku-bk21-\ud601\uc2e0\uc5f0\uad6c-\uc7a5\ud559\uae08",title:"Received the SKKU BK21 Innovative Research Scholarship (\uad50\uc721\uc5f0\uad6c\ub2e8 \uc7a5\ud559\ud601\uc2e0 (SKKU BK21 \ud601\uc2e0\uc5f0\uad6c \uc7a5\ud559\uae08))...",description:"",section:"News"},{id:"news-one-preprint-entitled-multi-modal-fusion-in-speech-emotion-recognition-a-comprehensive-review-of-methods-and-technologies-has-been-submitted-to-ssrn",title:"One preprint entitled \u201cMulti-modal fusion in speech emotion recognition: A comprehensive review of...",description:"",section:"News"},{id:"news-one-preprint-entitled-qtseg-a-query-token-based-architecture-for-efficient-2d-medical-image-segmentation-has-been-submitted-to-arxiv",title:"One preprint entitled \u201cQTSeg: A Query Token-Based Architecture for Efficient 2D Medical Image...",description:"",section:"News"},{id:"news-one-preprint-entitled-advances-in-rna-secondary-structure-prediction-and-rna-modifications-methods-data-and-applications-has-been-submitted-to-arxiv",title:"One preprint entitled \u201cAdvances in RNA secondary structure prediction and RNA modifications: Methods,...",description:"",section:"News"},{id:"news-one-collaborative-manuscript-entitled-enhancing-daily-runoff-prediction-in-hydropower-basins-with-a-voting-ensemble-model-using-historical-data-has-been-accepted-for-publication-in-the-hydrological-sciences-journal",title:"One collaborative manuscript entitled \u201cEnhancing daily runoff prediction in hydropower basins with a...",description:"",section:"News"},{id:"news-one-collaborative-manuscript-entitled-hydraulic-performance-and-wave-transmission-through-nature-inspired-perforated-hollow-base-piles-breakwater-has-been-accepted-for-publication-in-the-ocean-engineering-journal",title:"One collaborative manuscript entitled \u201cHydraulic performance and wave transmission through nature-inspired perforated hollow-base...",description:"",section:"News"},{id:"news-one-manuscript-entitled-dogpred-a-novel-deep-learning-framework-for-accurate-identification-of-human-o-linked-threonine-glycosylation-sites-has-been-accepted-for-publication-in-the-journal-of-molecular-biology",title:"One manuscript entitled \u201cDOGpred: A Novel Deep Learning Framework for Accurate Identification of...",description:"",section:"News"},{id:"news-one-collaborative-manuscript-entitled-memocmt-multimodal-emotion-recognition-using-cross-modal-transformer-based-feature-fusion-has-been-accepted-for-publication-in-the-scientific-reports-journal",title:"One collaborative manuscript entitled \u201cMemoCMT: multimodal emotion recognition using cross-modal transformer-based feature fusion\u201d...",description:"",section:"News"},{id:"news-received-the-caregen-chung-yong-ji-scholarship-\ucf00\uc5b4\uc820\uc815\uc6a9\uc9c0\uc7a5\ud559\uae08-\uc77c\ubc18\uc6d0",title:"Received the Caregen Chung Yong-ji Scholarship (\ucf00\uc5b4\uc820\uc815\uc6a9\uc9c0\uc7a5\ud559\uae08(\uc77c\ubc18\uc6d0))",description:"",section:"News"},{id:"news-received-the-university-innovation-awards-and-living-expenses-\ub300\ud559\ud601\uc2e0-\ud3ec\uc0c1\ubc0f\uc0dd\ud65c\ube44-\uc77c\ubc18\uc6d0",title:"Received the University Innovation - Awards and Living Expenses (\ub300\ud559\ud601\uc2e0-\ud3ec\uc0c1\ubc0f\uc0dd\ud65c\ube44(\uc77c\ubc18\uc6d0))",description:"",section:"News"},{id:"socials-email",title:"Send email",section:"Socials",handler:()=>{window.open("mailto:%70%68%61%6D%6E%68%61%74%74%72%75%6F%6E%67.%73%6B%79%6F@%67%6D%61%69%6C.%63%6F%6D","_blank")}},{id:"socials-orcid",title:"ORCID",section:"Socials",handler:()=>{window.open("https://orcid.org/0000-0002-8086-6722","_blank")}},{id:"socials-google-scholar",title:"Google Scholar",section:"Socials",handler:()=>{window.open("https://scholar.google.com/citations?user=HybH2XkAAAAJ","_blank")}},{id:"socials-semantic-scholar",title:"Semantic Scholar",section:"Socials",handler:()=>{window.open("https://www.semanticscholar.org/author/2077529712","_blank")}},{id:"socials-researchgate",title:"ResearchGate",section:"Socials",handler:()=>{window.open("https://www.researchgate.net/profile/Nhat-Truong-Pham/","_blank")}},{id:"socials-ieee-xplore",title:"IEEE Xplore",section:"Socials",handler:()=>{window.open("https://ieeexplore.ieee.org/author/37089489125/","_blank")}},{id:"socials-acm-dl",title:"ACM DL",section:"Socials",handler:()=>{window.open("https://dl.acm.org/profile/99660713417/","_blank")}},{id:"socials-scopus",title:"Scopus",section:"Socials",handler:()=>{window.open("https://www.scopus.com/authid/detail.uri?authorId=57243980400","_blank")}},{id:"socials-github",title:"GitHub",section:"Socials",handler:()=>{window.open("https://github.com/nhattruongpham","_blank")}},{id:"socials-linkedin",title:"LinkedIn",section:"Socials",handler:()=>{window.open("https://www.linkedin.com/in/nhattruongpham","_blank")}},{id:"socials-x",title:"X",description:"Twitter",section:"Socials",handler:()=>{window.open("https://twitter.com/nhattruong_pham","_blank")}},{id:"socials-dblp",title:"DBLP",section:"Socials",handler:()=>{window.open("https://dblp.org/pid/290/9204.html","_blank")}},{id:"socials-kaggle",title:"Kaggle",section:"Socials",handler:()=>{window.open("https://www.kaggle.com/nhattruongpham","_blank")}},{id:"socials-discord",title:"Discord",section:"Socials",handler:()=>{window.open("https://discord.com/users/673453370991837200","_blank")}},{id:"socials-zotero",title:"Zotero",section:"Socials",handler:()=>{window.open("https://www.zotero.org/nhattruongpham","_blank")}},{id:"light-theme",title:"Change theme to light",description:"Change the theme of the site to Light",section:"Theme",handler:()=>{setThemeSetting("light")}},{id:"dark-theme",title:"Change theme to dark",description:"Change the theme of the site to Dark",section:"Theme",handler:()=>{setThemeSetting("dark")}},{id:"system-theme",title:"Use system default theme",description:"Change the theme of the site to System Default",section:"Theme",handler:()=>{setThemeSetting("system")}}];</script> <script src="/assets/js/shortcut-key.js?6f508d74becd347268a7f822bca7309d"></script> </body> </html>